<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Capítulo 18 Bibliografía | Introducción a la Genética de Poblaciones y a la Genética Cuantitativa</title>
  <meta name="description" content="Apuntes del curso Genética II. Contenido teórico mínimo para el seguimiento del curso" />
  <meta name="generator" content="bookdown 0.34 and GitBook 2.6.7" />

  <meta property="og:title" content="Capítulo 18 Bibliografía | Introducción a la Genética de Poblaciones y a la Genética Cuantitativa" />
  <meta property="og:type" content="book" />
  
  <meta property="og:description" content="Apuntes del curso Genética II. Contenido teórico mínimo para el seguimiento del curso" />
  <meta name="github-repo" content="rstudio/bookdown-demo" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Capítulo 18 Bibliografía | Introducción a la Genética de Poblaciones y a la Genética Cuantitativa" />
  
  <meta name="twitter:description" content="Apuntes del curso Genética II. Contenido teórico mínimo para el seguimiento del curso" />
  

<meta name="author" content="Hugo Naya" />
<meta name="author" content="Federica Marín" />
<meta name="author" content="Mauricio Langleib" />
<meta name="author" content="(Jorge Urioste, María André, Andrea Larracharte, Washington Bell, Ana Laura Sánchez, Micaela Botta, Paula Batista, Rodrigo Vivián, Paola Gaiero))" />


<meta name="date" content="2023-09-20" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="apendicedos.html"/>

<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>
<script src="libs/kePrint-0.0.1/kePrint.js"></script>
<link href="libs/lightable-0.0.1/lightable.css" rel="stylesheet" />



<style type="text/css">
  
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
</style>
<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Genética II</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Prefacio</a>
<ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#nuestra-filosofía-del-no-tanto"><i class="fa fa-check"></i>Nuestra filosofía del NO (tanto)</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#qué-es-y-qué-no-es-este-libro"><i class="fa fa-check"></i>¿Qué ES y qué NO ES este libro?</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#bibliografía-recomendada-para-este-curso"><i class="fa fa-check"></i>Bibliografía recomendada para este curso</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#responsabilidades-y-agradecimientos"><i class="fa fa-check"></i>Responsabilidades y Agradecimientos</a></li>
</ul></li>
<li class="part"><span><b>Parte I: Genómica</b></span></li>
<li class="chapter" data-level="1" data-path="intro.html"><a href="intro.html"><i class="fa fa-check"></i><b>1</b> Introducción a la Genómica</a>
<ul>
<li class="chapter" data-level="1.1" data-path="intro.html"><a href="intro.html#variabilidad-genética"><i class="fa fa-check"></i><b>1.1</b> Variabilidad genética</a>
<ul>
<li class="chapter" data-level="" data-path="intro.html"><a href="intro.html#ejemplo-1.1"><i class="fa fa-check"></i>Ejemplo 1.1</a></li>
<li class="chapter" data-level="1.1.1" data-path="intro.html"><a href="intro.html#detectando-la-variabilidad-diferentes-técnicas"><i class="fa fa-check"></i><b>1.1.1</b> Detectando la variabilidad: diferentes técnicas</a></li>
<li class="chapter" data-level="1.1.2" data-path="intro.html"><a href="intro.html#metagenomica-y-metatranscriptomica"><i class="fa fa-check"></i><b>1.1.2</b> Metagenómica y Metatranscriptómica</a></li>
<li class="chapter" data-level="" data-path="intro.html"><a href="intro.html#ejemplo-1.2"><i class="fa fa-check"></i>Ejemplo 1.2</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="intro.html"><a href="intro.html#genómica-composicional"><i class="fa fa-check"></i><b>1.2</b> Genómica composicional</a>
<ul>
<li class="chapter" data-level="" data-path="intro.html"><a href="intro.html#contenido-gc-genómico-génico-correlaciones-y-textgc_textskew"><i class="fa fa-check"></i>Contenido GC genómico, génico, correlaciones y <span class="math inline">\(\text{GC}_{\text{skew}}\)</span></a></li>
<li class="chapter" data-level="" data-path="intro.html"><a href="intro.html#uso-de-codones"><i class="fa fa-check"></i>Uso de codones</a></li>
<li class="chapter" data-level="" data-path="intro.html"><a href="intro.html#ejemplo-1.3"><i class="fa fa-check"></i>Ejemplo 1.3</a></li>
<li class="chapter" data-level="1.2.1" data-path="intro.html"><a href="intro.html#exploracion-multivariada"><i class="fa fa-check"></i><b>1.2.1</b> Exploración multivariada: Análisis de Correspondencia</a></li>
<li class="chapter" data-level="1.2.2" data-path="intro.html"><a href="intro.html#composición-de-proteínas"><i class="fa fa-check"></i><b>1.2.2</b> Composición de proteínas</a></li>
<li class="chapter" data-level="" data-path="intro.html"><a href="intro.html#diferencias-en-propiedades-de-los-aas"><i class="fa fa-check"></i>Diferencias en propiedades de los AAs</a></li>
<li class="chapter" data-level="" data-path="intro.html"><a href="intro.html#diferencias-de-composición-entre-clases-de-proteínas"><i class="fa fa-check"></i>Diferencias de composición entre clases de proteínas</a></li>
<li class="chapter" data-level="" data-path="intro.html"><a href="intro.html#ejemplo-1.4"><i class="fa fa-check"></i>Ejemplo 1.4</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="intro.html"><a href="intro.html#genómica-comparativa"><i class="fa fa-check"></i><b>1.3</b> Genómica comparativa</a>
<ul>
<li class="chapter" data-level="" data-path="intro.html"><a href="intro.html#comparación-de-secuencias-génicas"><i class="fa fa-check"></i>Comparación de secuencias génicas</a></li>
<li class="chapter" data-level="" data-path="intro.html"><a href="intro.html#selección-positiva"><i class="fa fa-check"></i>Selección positiva</a></li>
<li class="chapter" data-level="" data-path="intro.html"><a href="intro.html#vías"><i class="fa fa-check"></i>Vías</a></li>
</ul></li>
<li class="chapter" data-level="1.4" data-path="intro.html"><a href="intro.html#genómica-funcional"><i class="fa fa-check"></i><b>1.4</b> Genómica funcional</a>
<ul>
<li class="chapter" data-level="" data-path="intro.html"><a href="intro.html#estudios-epigenéticos-y-epigenómicos"><i class="fa fa-check"></i>Estudios epigenéticos y epigenómicos</a></li>
</ul></li>
<li class="chapter" data-level="1.5" data-path="intro.html"><a href="intro.html#conclusión"><i class="fa fa-check"></i><b>1.5</b> Conclusión</a></li>
<li class="chapter" data-level="1.6" data-path="intro.html"><a href="intro.html#actividades"><i class="fa fa-check"></i><b>1.6</b> Actividades</a>
<ul>
<li class="chapter" data-level="1.6.1" data-path="intro.html"><a href="intro.html#control-de-lectura"><i class="fa fa-check"></i><b>1.6.1</b> Control de lectura</a></li>
<li class="chapter" data-level="1.6.2" data-path="intro.html"><a href="intro.html#verdadero-o-falso"><i class="fa fa-check"></i><b>1.6.2</b> ¿Verdadero o falso?</a></li>
<li class="chapter" data-level="" data-path="intro.html"><a href="intro.html#ejercicios-1"><i class="fa fa-check"></i>Ejercicios</a></li>
</ul></li>
</ul></li>
<li class="part"><span><b>Parte II: Genética de Poblaciones</b></span></li>
<li class="chapter" data-level="2" data-path="variacion.html"><a href="variacion.html"><i class="fa fa-check"></i><b>2</b> Variación y equilibrio de Hardy-Weinberg</a>
<ul>
<li class="chapter" data-level="2.1" data-path="variacion.html"><a href="variacion.html#el-equilibrio-de-hardy-weinberg"><i class="fa fa-check"></i><b>2.1</b> El equilibrio de Hardy-Weinberg</a>
<ul>
<li class="chapter" data-level="" data-path="variacion.html"><a href="variacion.html#supuestos-que-asumimos-se-cumplen-para-h-w"><i class="fa fa-check"></i>Supuestos que asumimos se cumplen para H-W</a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="variacion.html"><a href="variacion.html#hardy-weinberg-en-especies-dioicas-dos-sexos"><i class="fa fa-check"></i><b>2.2</b> Hardy-Weinberg en especies dioicas (dos sexos)</a></li>
<li class="chapter" data-level="2.3" data-path="variacion.html"><a href="variacion.html#heterocigotas-freq-alelica"><i class="fa fa-check"></i><b>2.3</b> H-W: la frecuencia de heterocigotas en función de la frecuencia alélica</a></li>
<li class="chapter" data-level="2.4" data-path="variacion.html"><a href="variacion.html#el-equilibrio-de-hardy-weinberg-en-cromosomas-ligados-al-sexo"><i class="fa fa-check"></i><b>2.4</b> El equilibrio de Hardy-Weinberg en cromosomas ligados al sexo</a></li>
<li class="chapter" data-level="2.5" data-path="variacion.html"><a href="variacion.html#tres-o-más-alelos"><i class="fa fa-check"></i><b>2.5</b> Tres o más alelos</a></li>
<li class="chapter" data-level="2.6" data-path="variacion.html"><a href="variacion.html#la-estimación-de-frecuencias-y-el-equilibrio-o-no"><i class="fa fa-check"></i><b>2.6</b> La estimación de frecuencias y el equilibrio (o no)</a></li>
<li class="chapter" data-level="2.7" data-path="variacion.html"><a href="variacion.html#el-sistema-abo"><i class="fa fa-check"></i><b>2.7</b> El sistema ABO</a></li>
<li class="chapter" data-level="2.8" data-path="variacion.html"><a href="variacion.html#dónde-se-esconden-los-alelos-recesivos"><i class="fa fa-check"></i><b>2.8</b> ¿Dónde se “esconden” los alelos recesivos?</a></li>
<li class="chapter" data-level="2.9" data-path="variacion.html"><a href="variacion.html#hardy-weinberg-en-especies-poliploides"><i class="fa fa-check"></i><b>2.9</b> Hardy-Weinberg en especies poliploides</a></li>
<li class="chapter" data-level="2.10" data-path="variacion.html"><a href="variacion.html#geometría-y-genética-los-diagramas-de-de-finetti"><i class="fa fa-check"></i><b>2.10</b> Geometría y Genética: los diagramas de <em>de Finetti</em></a></li>
<li class="chapter" data-level="2.11" data-path="variacion.html"><a href="variacion.html#estimacion-abo"><i class="fa fa-check"></i><b>2.11</b> La estimación de frecuencias en el locus ABO</a></li>
<li class="chapter" data-level="2.12" data-path="variacion.html"><a href="variacion.html#conclusión-1"><i class="fa fa-check"></i><b>2.12</b> Conclusión</a></li>
<li class="chapter" data-level="2.13" data-path="variacion.html"><a href="variacion.html#actividades-1"><i class="fa fa-check"></i><b>2.13</b> Actividades</a>
<ul>
<li class="chapter" data-level="2.13.1" data-path="variacion.html"><a href="variacion.html#control-de-lectura-1"><i class="fa fa-check"></i><b>2.13.1</b> Control de lectura</a></li>
<li class="chapter" data-level="2.13.2" data-path="variacion.html"><a href="variacion.html#verdadero-o-falso-1"><i class="fa fa-check"></i><b>2.13.2</b> ¿Verdadero o falso?</a></li>
<li class="chapter" data-level="" data-path="variacion.html"><a href="variacion.html#ejercicios-2"><i class="fa fa-check"></i>Ejercicios</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="3" data-path="deriva.html"><a href="deriva.html"><i class="fa fa-check"></i><b>3</b> Deriva Genética</a>
<ul>
<li class="chapter" data-level="3.1" data-path="deriva.html"><a href="deriva.html#el-rol-de-los-procesos-estocásticos-en-la-genética"><i class="fa fa-check"></i><b>3.1</b> El rol de los procesos estocásticos en la genética</a></li>
<li class="chapter" data-level="3.2" data-path="deriva.html"><a href="deriva.html#el-modelo-de-wright-fisher"><i class="fa fa-check"></i><b>3.2</b> El modelo de Wright-Fisher</a>
<ul>
<li class="chapter" data-level="" data-path="deriva.html"><a href="deriva.html#ejemplo-3.1"><i class="fa fa-check"></i>Ejemplo 3.1</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="deriva.html"><a href="deriva.html#el-rol-de-la-subdivisión-poblacional-en-la-evolución-de-las-frecuencias-alélicas"><i class="fa fa-check"></i><b>3.3</b> El rol de la subdivisión poblacional en la evolución de las frecuencias alélicas</a>
<ul>
<li class="chapter" data-level="" data-path="deriva.html"><a href="deriva.html#homocigosidad-y-heterocigosidad"><i class="fa fa-check"></i>Homocigosidad y Heterocigosidad</a></li>
<li class="chapter" data-level="" data-path="deriva.html"><a href="deriva.html#ejemplo-3.2"><i class="fa fa-check"></i>Ejemplo 3.2</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="deriva.html"><a href="deriva.html#cadenas-de-markov"><i class="fa fa-check"></i><b>3.4</b> Cadenas de Markov</a></li>
<li class="chapter" data-level="3.5" data-path="deriva.html"><a href="deriva.html#tamaño-efectivo-poblacional"><i class="fa fa-check"></i><b>3.5</b> Tamaño efectivo poblacional</a>
<ul>
<li class="chapter" data-level="" data-path="deriva.html"><a href="deriva.html#ejemplo-3.3"><i class="fa fa-check"></i>Ejemplo 3.3</a></li>
<li class="chapter" data-level="" data-path="deriva.html"><a href="deriva.html#ejemplo-3.4"><i class="fa fa-check"></i>Ejemplo 3.4</a></li>
</ul></li>
<li class="chapter" data-level="3.6" data-path="deriva.html"><a href="deriva.html#aproximación-de-difusión"><i class="fa fa-check"></i><b>3.6</b> Aproximación de difusión</a>
<ul>
<li class="chapter" data-level="" data-path="deriva.html"><a href="deriva.html#caminatas-al-azar-y-procesos-de-difusión"><i class="fa fa-check"></i>Caminatas al azar y procesos de difusión</a></li>
<li class="chapter" data-level="" data-path="deriva.html"><a href="deriva.html#kolmogorov-forward-equation"><i class="fa fa-check"></i>Kolmogorov Forward Equation</a></li>
<li class="chapter" data-level="" data-path="deriva.html"><a href="deriva.html#kolmogorov-backward-equation"><i class="fa fa-check"></i>Kolmogorov Backward Equation</a></li>
<li class="chapter" data-level="" data-path="deriva.html"><a href="deriva.html#solución-de-las-ecuaciones"><i class="fa fa-check"></i>Solución de las ecuaciones</a></li>
</ul></li>
<li class="chapter" data-level="3.7" data-path="deriva.html"><a href="deriva.html#probabilidad-de-fijación-y-tiempos-de-fijación"><i class="fa fa-check"></i><b>3.7</b> Probabilidad de fijación y tiempos de fijación</a>
<ul>
<li class="chapter" data-level="" data-path="deriva.html"><a href="deriva.html#ejemplo-3.5"><i class="fa fa-check"></i>Ejemplo 3.5</a></li>
</ul></li>
<li class="chapter" data-level="3.8" data-path="deriva.html"><a href="deriva.html#el-modelo-coalescente"><i class="fa fa-check"></i><b>3.8</b> El modelo coalescente</a></li>
<li class="chapter" data-level="3.9" data-path="deriva.html"><a href="deriva.html#conclusión-2"><i class="fa fa-check"></i><b>3.9</b> Conclusión</a></li>
<li class="chapter" data-level="3.10" data-path="deriva.html"><a href="deriva.html#actividades-2"><i class="fa fa-check"></i><b>3.10</b> Actividades</a>
<ul>
<li class="chapter" data-level="3.10.1" data-path="deriva.html"><a href="deriva.html#control-de-lectura-2"><i class="fa fa-check"></i><b>3.10.1</b> Control de lectura</a></li>
<li class="chapter" data-level="3.10.2" data-path="deriva.html"><a href="deriva.html#verdadero-o-falso-2"><i class="fa fa-check"></i><b>3.10.2</b> ¿Verdadero o falso?</a></li>
<li class="chapter" data-level="3.10.3" data-path="deriva.html"><a href="deriva.html#ejercicios"><i class="fa fa-check"></i><b>3.10.3</b> Ejercicios</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="seleccion.html"><a href="seleccion.html"><i class="fa fa-check"></i><b>4</b> Selección Natural</a>
<ul>
<li class="chapter" data-level="4.1" data-path="seleccion.html"><a href="seleccion.html#el-concepto-de-fitness"><i class="fa fa-check"></i><b>4.1</b> El concepto de “fitness”</a></li>
<li class="chapter" data-level="4.2" data-path="seleccion.html"><a href="seleccion.html#selección-natural-en-el-modelo-de-un-locus-con-dos-alelos"><i class="fa fa-check"></i><b>4.2</b> Selección natural en el modelo de un locus con dos alelos</a></li>
<li class="chapter" data-level="4.3" data-path="seleccion.html"><a href="seleccion.html#diferentes-formas-de-selección"><i class="fa fa-check"></i><b>4.3</b> Diferentes formas de selección</a>
<ul>
<li class="chapter" data-level="4.3.1" data-path="seleccion.html"><a href="seleccion.html#selección-direccional"><i class="fa fa-check"></i><b>4.3.1</b> Selección direccional</a></li>
<li class="chapter" data-level="4.3.2" data-path="seleccion.html"><a href="seleccion.html#selección-estabilizadora"><i class="fa fa-check"></i><b>4.3.2</b> Selección estabilizadora</a></li>
<li class="chapter" data-level="4.3.3" data-path="seleccion.html"><a href="seleccion.html#selección-disruptiva"><i class="fa fa-check"></i><b>4.3.3</b> Selección disruptiva</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="seleccion.html"><a href="seleccion.html#el-teorema-fundamental-de-la-selección-natural"><i class="fa fa-check"></i><b>4.4</b> El teorema fundamental de la selección natural</a></li>
<li class="chapter" data-level="4.5" data-path="seleccion.html"><a href="seleccion.html#equilibrio-selección-mutación"><i class="fa fa-check"></i><b>4.5</b> Equilibrio selección-mutación</a>
<ul>
<li class="chapter" data-level="4.5.1" data-path="seleccion.html"><a href="seleccion.html#el-principio-de-haldane-muller"><i class="fa fa-check"></i><b>4.5.1</b> El principio de Haldane-Muller</a></li>
</ul></li>
<li class="chapter" data-level="4.6" data-path="seleccion.html"><a href="seleccion.html#la-fuerza-de-la-selección-natural"><i class="fa fa-check"></i><b>4.6</b> La fuerza de la selección natural</a></li>
<li class="chapter" data-level="4.7" data-path="seleccion.html"><a href="seleccion.html#equilibrio-selección-deriva"><i class="fa fa-check"></i><b>4.7</b> Equilibrio selección-deriva</a></li>
<li class="chapter" data-level="4.8" data-path="seleccion.html"><a href="seleccion.html#otros-tipos-de-selección-y-complejidades"><i class="fa fa-check"></i><b>4.8</b> Otros tipos de selección y complejidades</a></li>
<li class="chapter" data-level="4.9" data-path="seleccion.html"><a href="seleccion.html#conclusión-3"><i class="fa fa-check"></i><b>4.9</b> Conclusión</a></li>
<li class="chapter" data-level="4.10" data-path="seleccion.html"><a href="seleccion.html#actividades-3"><i class="fa fa-check"></i><b>4.10</b> Actividades</a>
<ul>
<li class="chapter" data-level="4.10.1" data-path="seleccion.html"><a href="seleccion.html#control-de-lectura-3"><i class="fa fa-check"></i><b>4.10.1</b> Control de lectura</a></li>
<li class="chapter" data-level="4.10.2" data-path="seleccion.html"><a href="seleccion.html#verdadero-o-falso-3"><i class="fa fa-check"></i><b>4.10.2</b> ¿Verdadero o falso?</a></li>
<li class="chapter" data-level="" data-path="seleccion.html"><a href="seleccion.html#ejercicios-4"><i class="fa fa-check"></i>Ejercicios</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="dosloci.html"><a href="dosloci.html"><i class="fa fa-check"></i><b>5</b> Dinámica de 2 <em>loci</em></a>
<ul>
<li class="chapter" data-level="5.1" data-path="dosloci.html"><a href="dosloci.html#desequilibrio-de-ligamiento-y-recombinación"><i class="fa fa-check"></i><b>5.1</b> Desequilibrio de ligamiento y recombinación</a></li>
<li class="chapter" data-level="5.2" data-path="dosloci.html"><a href="dosloci.html#la-evolución-en-el-tiempo-del-desequilibrio-de-ligamiento"><i class="fa fa-check"></i><b>5.2</b> La evolución en el tiempo del desequilibrio de ligamiento</a></li>
<li class="chapter" data-level="5.3" data-path="dosloci.html"><a href="dosloci.html#otras-medidas-de-asociación"><i class="fa fa-check"></i><b>5.3</b> Otras medidas de asociación</a>
<ul>
<li class="chapter" data-level="" data-path="dosloci.html"><a href="dosloci.html#ejemplo-5.3"><i class="fa fa-check"></i>Ejemplo 5.3</a></li>
</ul></li>
<li class="chapter" data-level="5.4" data-path="dosloci.html"><a href="dosloci.html#selección-en-modelos-de-dos-loci"><i class="fa fa-check"></i><b>5.4</b> Selección en modelos de dos <em>loci</em></a>
<ul>
<li class="chapter" data-level="" data-path="dosloci.html"><a href="dosloci.html#selección-en-un-locus-impacto-en-loci-en-desequilibrio-de-ligamiento"><i class="fa fa-check"></i>Selección en un locus: impacto en <em>loci</em> en desequilibrio de ligamiento</a></li>
</ul></li>
<li class="chapter" data-level="5.5" data-path="dosloci.html"><a href="dosloci.html#arrastre-genético-genetic-hitchhiking-o-genetic-draft"><i class="fa fa-check"></i><b>5.5</b> Arrastre genético (“<em>genetic hitchhiking</em>” o “<em>genetic draft</em>”)</a></li>
<li class="chapter" data-level="5.6" data-path="dosloci.html"><a href="dosloci.html#causas-del-desequilibrio-de-ligamiento"><i class="fa fa-check"></i><b>5.6</b> Causas del desequilibrio de ligamiento</a>
<ul>
<li class="chapter" data-level="" data-path="dosloci.html"><a href="dosloci.html#desequilibrio-debido-al-mestizaje"><i class="fa fa-check"></i>Desequilibrio debido al mestizaje</a></li>
<li class="chapter" data-level="" data-path="dosloci.html"><a href="dosloci.html#desequilibrio-inducido-por-el-sistema-de-apareamiento"><i class="fa fa-check"></i>Desequilibrio inducido por el sistema de apareamiento</a></li>
<li class="chapter" data-level="" data-path="dosloci.html"><a href="dosloci.html#el-cromosoma-y"><i class="fa fa-check"></i>El cromosoma Y</a></li>
<li class="chapter" data-level="" data-path="dosloci.html"><a href="dosloci.html#desequilibrio-debido-a-recombinación-reducida"><i class="fa fa-check"></i>Desequilibrio debido a recombinación reducida</a></li>
</ul></li>
<li class="chapter" data-level="5.7" data-path="dosloci.html"><a href="dosloci.html#conclusión-4"><i class="fa fa-check"></i><b>5.7</b> Conclusión</a></li>
<li class="chapter" data-level="5.8" data-path="dosloci.html"><a href="dosloci.html#actividades-4"><i class="fa fa-check"></i><b>5.8</b> Actividades</a>
<ul>
<li class="chapter" data-level="5.8.1" data-path="dosloci.html"><a href="dosloci.html#control-de-lectura-4"><i class="fa fa-check"></i><b>5.8.1</b> Control de lectura</a></li>
<li class="chapter" data-level="5.8.2" data-path="dosloci.html"><a href="dosloci.html#verdadero-o-falso-4"><i class="fa fa-check"></i><b>5.8.2</b> ¿Verdadero o falso?</a></li>
<li class="chapter" data-level="" data-path="dosloci.html"><a href="dosloci.html#ejercicios-5"><i class="fa fa-check"></i>Ejercicios</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="aparnoalazar.html"><a href="aparnoalazar.html"><i class="fa fa-check"></i><b>6</b> Apareamientos no-aleatorios</a>
<ul>
<li class="chapter" data-level="6.1" data-path="aparnoalazar.html"><a href="aparnoalazar.html#el-concepto-de-identidad-por-ascendencia-ibd"><i class="fa fa-check"></i><b>6.1</b> El concepto de “identidad por ascendencia” (IBD)</a></li>
<li class="chapter" data-level="6.2" data-path="aparnoalazar.html"><a href="aparnoalazar.html#generalización-de-hardy-weinberg-para-apareamientos-no-aleatorios"><i class="fa fa-check"></i><b>6.2</b> Generalización de Hardy-Weinberg para apareamientos no-aleatorios</a></li>
<li class="chapter" data-level="6.3" data-path="aparnoalazar.html"><a href="aparnoalazar.html#f-como-correlación-entre-gametos-unidos"><i class="fa fa-check"></i><b>6.3</b> <span class="math inline">\(F\)</span> como correlación entre gametos unidos</a></li>
<li class="chapter" data-level="6.4" data-path="aparnoalazar.html"><a href="aparnoalazar.html#endocría-y-depresión-endogámica"><i class="fa fa-check"></i><b>6.4</b> Endocría y depresión endogámica</a></li>
<li class="chapter" data-level="6.5" data-path="aparnoalazar.html"><a href="aparnoalazar.html#un-caso-extremo-la-autogamia"><i class="fa fa-check"></i><b>6.5</b> Un caso extremo: la autogamia</a></li>
<li class="chapter" data-level="6.6" data-path="aparnoalazar.html"><a href="aparnoalazar.html#el-coeficiente-de-endocría-y-los-estadísticos-f"><i class="fa fa-check"></i><b>6.6</b> El coeficiente de endocría y los estadísticos <em>F</em></a>
<ul>
<li class="chapter" data-level="" data-path="aparnoalazar.html"><a href="aparnoalazar.html#ejemplo-6.1"><i class="fa fa-check"></i>Ejemplo 6.1</a></li>
</ul></li>
<li class="chapter" data-level="6.7" data-path="aparnoalazar.html"><a href="aparnoalazar.html#el-efecto-wahlund"><i class="fa fa-check"></i><b>6.7</b> El efecto Wahlund</a>
<ul>
<li class="chapter" data-level="" data-path="aparnoalazar.html"><a href="aparnoalazar.html#ejemplo-6.2"><i class="fa fa-check"></i>Ejemplo 6.2</a></li>
</ul></li>
<li class="chapter" data-level="6.8" data-path="aparnoalazar.html"><a href="aparnoalazar.html#subdivisión-migración-y-el-modelo-de-islas"><i class="fa fa-check"></i><b>6.8</b> Subdivisión, migración y el modelo de islas</a>
<ul>
<li class="chapter" data-level="" data-path="aparnoalazar.html"><a href="aparnoalazar.html#ejemplo-6.3"><i class="fa fa-check"></i>Ejemplo 6.3</a></li>
</ul></li>
<li class="chapter" data-level="6.9" data-path="aparnoalazar.html"><a href="aparnoalazar.html#mecanismos-de-especiación"><i class="fa fa-check"></i><b>6.9</b> Mecanismos de especiación</a></li>
<li class="chapter" data-level="6.10" data-path="aparnoalazar.html"><a href="aparnoalazar.html#conclusión-5"><i class="fa fa-check"></i><b>6.10</b> Conclusión</a></li>
<li class="chapter" data-level="6.11" data-path="aparnoalazar.html"><a href="aparnoalazar.html#actividades-5"><i class="fa fa-check"></i><b>6.11</b> Actividades</a>
<ul>
<li class="chapter" data-level="6.11.1" data-path="aparnoalazar.html"><a href="aparnoalazar.html#control-de-lectura-5"><i class="fa fa-check"></i><b>6.11.1</b> Control de lectura</a></li>
<li class="chapter" data-level="6.11.2" data-path="aparnoalazar.html"><a href="aparnoalazar.html#verdadero-o-falso-5"><i class="fa fa-check"></i><b>6.11.2</b> ¿Verdadero o falso?</a></li>
<li class="chapter" data-level="" data-path="aparnoalazar.html"><a href="aparnoalazar.html#ejercicios-6"><i class="fa fa-check"></i>Ejercicios</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="7" data-path="microbial.html"><a href="microbial.html"><i class="fa fa-check"></i><b>7</b> Genética de Poblaciones Microbianas</a>
<ul>
<li class="chapter" data-level="7.1" data-path="microbial.html"><a href="microbial.html#genómica-y-mecanismos-de-herencia-en-procariotas"><i class="fa fa-check"></i><b>7.1</b> Genómica y mecanismos de herencia en procariotas</a></li>
<li class="chapter" data-level="7.2" data-path="microbial.html"><a href="microbial.html#dinámica-de-las-poblaciones-bacterianas"><i class="fa fa-check"></i><b>7.2</b> Dinámica de las poblaciones bacterianas</a>
<ul>
<li class="chapter" data-level="" data-path="microbial.html"><a href="microbial.html#el-modelo-exponencial"><i class="fa fa-check"></i>El modelo exponencial</a></li>
<li class="chapter" data-level="" data-path="microbial.html"><a href="microbial.html#ejemplo-7.1"><i class="fa fa-check"></i>Ejemplo 7.1</a></li>
<li class="chapter" data-level="" data-path="microbial.html"><a href="microbial.html#el-modelo-logístico"><i class="fa fa-check"></i>El modelo logístico</a></li>
<li class="chapter" data-level="" data-path="microbial.html"><a href="microbial.html#ejemplo-7.2"><i class="fa fa-check"></i>Ejemplo 7.2</a></li>
</ul></li>
<li class="chapter" data-level="7.3" data-path="microbial.html"><a href="microbial.html#modelos-haploides-de-selección-natural"><i class="fa fa-check"></i><b>7.3</b> Modelos haploides de selección natural</a>
<ul>
<li class="chapter" data-level="" data-path="microbial.html"><a href="microbial.html#selección-haploide-modelo-discreto"><i class="fa fa-check"></i>Selección haploide: modelo discreto</a></li>
<li class="chapter" data-level="" data-path="microbial.html"><a href="microbial.html#ejemplo-7.3"><i class="fa fa-check"></i>Ejemplo 7.3</a></li>
<li class="chapter" data-level="" data-path="microbial.html"><a href="microbial.html#selección-haploide-modelo-continuo"><i class="fa fa-check"></i>Selección haploide: modelo continuo</a></li>
<li class="chapter" data-level="" data-path="microbial.html"><a href="microbial.html#ejemplo-7.4"><i class="fa fa-check"></i>Ejemplo 7.4</a></li>
</ul></li>
<li class="chapter" data-level="7.4" data-path="microbial.html"><a href="microbial.html#los-modelos-de-moran-y-de-fisión-vs-el-de-wright-fisher"><i class="fa fa-check"></i><b>7.4</b> Los modelos de Moran y de fisión <em>vs</em> el de Wright-Fisher</a></li>
<li class="chapter" data-level="7.5" data-path="microbial.html"><a href="microbial.html#el-rol-de-la-transferencia-horizontal"><i class="fa fa-check"></i><b>7.5</b> El rol de la transferencia horizontal</a></li>
<li class="chapter" data-level="7.6" data-path="microbial.html"><a href="microbial.html#seleccionismo-vs-neutralismo-los-procariotas-en-el-debate"><i class="fa fa-check"></i><b>7.6</b> Seleccionismo <em>vs</em> neutralismo: los procariotas en el debate</a></li>
<li class="chapter" data-level="7.7" data-path="microbial.html"><a href="microbial.html#genómica-poblacional"><i class="fa fa-check"></i><b>7.7</b> Genómica poblacional</a></li>
<li class="chapter" data-level="7.8" data-path="microbial.html"><a href="microbial.html#genes-de-resistencia"><i class="fa fa-check"></i><b>7.8</b> Genes de resistencia</a></li>
<li class="chapter" data-level="7.9" data-path="microbial.html"><a href="microbial.html#introducción-a-la-epidemiología-modelos-compartimentales"><i class="fa fa-check"></i><b>7.9</b> Introducción a la epidemiología: modelos compartimentales</a></li>
<li class="chapter" data-level="7.10" data-path="microbial.html"><a href="microbial.html#conclusión-6"><i class="fa fa-check"></i><b>7.10</b> Conclusión</a></li>
<li class="chapter" data-level="7.11" data-path="microbial.html"><a href="microbial.html#actividades-6"><i class="fa fa-check"></i><b>7.11</b> Actividades</a>
<ul>
<li class="chapter" data-level="7.11.1" data-path="microbial.html"><a href="microbial.html#control-de-lectura-6"><i class="fa fa-check"></i><b>7.11.1</b> Control de lectura</a></li>
<li class="chapter" data-level="7.11.2" data-path="microbial.html"><a href="microbial.html#verdadero-o-falso-6"><i class="fa fa-check"></i><b>7.11.2</b> ¿Verdadero o falso?</a></li>
<li class="chapter" data-level="" data-path="microbial.html"><a href="microbial.html#ejercicios-7"><i class="fa fa-check"></i>Ejercicios</a></li>
</ul></li>
</ul></li>
<li class="part"><span><b>Parte III: Genética Cuantitativa</b></span></li>
<li class="chapter" data-level="8" data-path="modelogenbas.html"><a href="modelogenbas.html"><i class="fa fa-check"></i><b>8</b> El Modelo Genético Básico</a>
<ul>
<li class="chapter" data-level="8.1" data-path="modelogenbas.html"><a href="modelogenbas.html#variación-continua-y-discreta"><i class="fa fa-check"></i><b>8.1</b> Variación continua y discreta</a></li>
<li class="chapter" data-level="8.2" data-path="modelogenbas.html"><a href="modelogenbas.html#el-modelo-genético-básico"><i class="fa fa-check"></i><b>8.2</b> El modelo genético básico</a></li>
<li class="chapter" data-level="8.3" data-path="modelogenbas.html"><a href="modelogenbas.html#modelo-genético-básico-un-locus-con-dos-alelos"><i class="fa fa-check"></i><b>8.3</b> Modelo Genético Básico: un <em>locus</em> con dos alelos</a>
<ul>
<li class="chapter" data-level="" data-path="modelogenbas.html"><a href="modelogenbas.html#media-de-la-población"><i class="fa fa-check"></i>Media de la población</a></li>
</ul></li>
<li class="chapter" data-level="8.4" data-path="modelogenbas.html"><a href="modelogenbas.html#efecto-medio"><i class="fa fa-check"></i><b>8.4</b> Efecto medio</a></li>
<li class="chapter" data-level="8.5" data-path="modelogenbas.html"><a href="modelogenbas.html#valor-reproductivo-o-valor-de-cría"><i class="fa fa-check"></i><b>8.5</b> Valor reproductivo (o valor de cría)</a>
<ul>
<li class="chapter" data-level="" data-path="modelogenbas.html"><a href="modelogenbas.html#derivación-de-los-efectos-medios-por-mínimos-cuadrados"><i class="fa fa-check"></i>Derivación de los efectos medios por mínimos cuadrados</a></li>
</ul></li>
<li class="chapter" data-level="8.6" data-path="modelogenbas.html"><a href="modelogenbas.html#desvío-de-dominancia"><i class="fa fa-check"></i><b>8.6</b> Desvío de dominancia</a></li>
<li class="chapter" data-level="8.7" data-path="modelogenbas.html"><a href="modelogenbas.html#interacción-genotipo-x-ambiente"><i class="fa fa-check"></i><b>8.7</b> Interacción Genotipo x Ambiente</a></li>
<li class="chapter" data-level="8.8" data-path="modelogenbas.html"><a href="modelogenbas.html#la-varianza-en-el-modelo-genético-básico"><i class="fa fa-check"></i><b>8.8</b> La Varianza en el modelo genético básico</a>
<ul>
<li class="chapter" data-level="" data-path="modelogenbas.html"><a href="modelogenbas.html#componentes-genéticos-de-la-varianza-un-locus-con-dos-alelos"><i class="fa fa-check"></i>Componentes genéticos de la varianza: un <em>locus</em> con dos alelos</a></li>
<li class="chapter" data-level="" data-path="modelogenbas.html"><a href="modelogenbas.html#la-varianza-ambiental"><i class="fa fa-check"></i>La varianza ambiental</a></li>
<li class="chapter" data-level="" data-path="modelogenbas.html"><a href="modelogenbas.html#la-varianza-de-la-interacción"><i class="fa fa-check"></i>La Varianza de la interacción</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="modelogenbas.html"><a href="modelogenbas.html#ejercicios-8"><i class="fa fa-check"></i>Ejercicios</a></li>
</ul></li>
<li class="chapter" data-level="9" data-path="parentesco.html"><a href="parentesco.html"><i class="fa fa-check"></i><b>9</b> Parentesco y Semejanza entre Parientes</a>
<ul>
<li class="chapter" data-level="9.1" data-path="parentesco.html"><a href="parentesco.html#parentesco-1"><i class="fa fa-check"></i><b>9.1</b> Parentesco</a></li>
<li class="chapter" data-level="9.2" data-path="parentesco.html"><a href="parentesco.html#parentesco-aditivo"><i class="fa fa-check"></i><b>9.2</b> Parentesco aditivo</a>
<ul>
<li class="chapter" data-level="" data-path="parentesco.html"><a href="parentesco.html#consanguinidad"><i class="fa fa-check"></i>Consanguinidad</a></li>
<li class="chapter" data-level="" data-path="parentesco.html"><a href="parentesco.html#diagramas-de-flechas"><i class="fa fa-check"></i>Diagramas de flechas</a></li>
<li class="chapter" data-level="" data-path="parentesco.html"><a href="parentesco.html#el-método-tabular"><i class="fa fa-check"></i>El método tabular</a></li>
</ul></li>
<li class="chapter" data-level="9.3" data-path="parentesco.html"><a href="parentesco.html#parentesco-de-dominancia"><i class="fa fa-check"></i><b>9.3</b> Parentesco de dominancia</a></li>
<li class="chapter" data-level="9.4" data-path="parentesco.html"><a href="parentesco.html#semejanza-entre-parientes"><i class="fa fa-check"></i><b>9.4</b> Semejanza entre parientes</a></li>
<li class="chapter" data-level="9.5" data-path="parentesco.html"><a href="parentesco.html#estimación-de-las-varianzas-aditiva-y-de-dominancia"><i class="fa fa-check"></i><b>9.5</b> Estimación de las varianzas aditiva y de dominancia</a>
<ul>
<li class="chapter" data-level="" data-path="parentesco.html"><a href="parentesco.html#propiedades-deseables-de-los-estimadores"><i class="fa fa-check"></i>Propiedades deseables de los estimadores</a></li>
<li class="chapter" data-level="" data-path="parentesco.html"><a href="parentesco.html#estimación-de-la-varianza-aditiva-a-partir-del-análisis-de-varianza-anova"><i class="fa fa-check"></i>Estimación de la varianza aditiva a partir del análisis de varianza (ANOVA)</a></li>
<li class="chapter" data-level="" data-path="parentesco.html"><a href="parentesco.html#estimaciones-basadas-en-modelos-mixtos-lineales"><i class="fa fa-check"></i>Estimaciones basadas en modelos mixtos lineales</a></li>
</ul></li>
<li class="chapter" data-level="9.6" data-path="parentesco.html"><a href="parentesco.html#parentesco-genómico"><i class="fa fa-check"></i><b>9.6</b> Parentesco genómico</a>
<ul>
<li class="chapter" data-level="" data-path="parentesco.html"><a href="parentesco.html#contenido-génico"><i class="fa fa-check"></i>Contenido génico</a></li>
<li class="chapter" data-level="" data-path="parentesco.html"><a href="parentesco.html#la-matriz-de-similaridad-genómica"><i class="fa fa-check"></i>La matriz de similaridad genómica</a></li>
</ul></li>
<li class="chapter" data-level="9.7" data-path="parentesco.html"><a href="parentesco.html#estudios-de-ascendencia-ancestría"><i class="fa fa-check"></i><b>9.7</b> Estudios de ascendencia (“ancestría”)</a></li>
<li class="chapter" data-level="" data-path="parentesco.html"><a href="parentesco.html#ejercicios-9"><i class="fa fa-check"></i>Ejercicios</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="pargen.html"><a href="pargen.html"><i class="fa fa-check"></i><b>10</b> Parámetros Genéticos: Heredabilidad y Repetibilidad</a>
<ul>
<li class="chapter" data-level="10.1" data-path="pargen.html"><a href="pargen.html#heredabilidad"><i class="fa fa-check"></i><b>10.1</b> Heredabilidad</a>
<ul>
<li class="chapter" data-level="" data-path="pargen.html"><a href="pargen.html#la-heredabilidad-como-relación-de-varianzas"><i class="fa fa-check"></i>La heredabilidad como relación de varianzas</a></li>
<li class="chapter" data-level="" data-path="pargen.html"><a href="pargen.html#la-heredabilidad-como-regresión"><i class="fa fa-check"></i>La heredabilidad como regresión</a></li>
<li class="chapter" data-level="" data-path="pargen.html"><a href="pargen.html#regresión-del-valor-de-cría-en-el-fenotipo-individual"><i class="fa fa-check"></i>Regresión del valor de cría en el fenotipo individual</a></li>
<li class="chapter" data-level="" data-path="pargen.html"><a href="pargen.html#regresión-del-valor-fenotípico-de-los-hijos-sobre-el-fenotipo-de-un-progenitor"><i class="fa fa-check"></i>Regresión del valor fenotípico de los hijos sobre el fenotipo de un progenitor</a></li>
<li class="chapter" data-level="" data-path="pargen.html"><a href="pargen.html#regresión-del-valor-fenotípico-de-los-hijos-sobre-el-fenotipo-de-ambos-progenitores"><i class="fa fa-check"></i>Regresión del valor fenotípico de los hijos sobre el fenotipo de ambos progenitores</a></li>
<li class="chapter" data-level="" data-path="pargen.html"><a href="pargen.html#precisión-en-las-estimaciones-de-heredabilidad-varianza-del-estimador"><i class="fa fa-check"></i>Precisión en las estimaciones de heredabilidad: varianza del estimador</a></li>
</ul></li>
<li class="chapter" data-level="10.2" data-path="pargen.html"><a href="pargen.html#heredabilidad-en-sentido-amplio-y-sentido-estricto"><i class="fa fa-check"></i><b>10.2</b> Heredabilidad en sentido amplio y sentido estricto</a></li>
<li class="chapter" data-level="10.3" data-path="pargen.html"><a href="pargen.html#heredabilidad-lograda"><i class="fa fa-check"></i><b>10.3</b> Heredabilidad lograda</a></li>
<li class="chapter" data-level="10.4" data-path="pargen.html"><a href="pargen.html#heredabilidad-en-poblaciones-agronómicas-y-de-laboratorio-vs-poblaciones-naturales"><i class="fa fa-check"></i><b>10.4</b> Heredabilidad en poblaciones agronómicas y de laboratorio vs poblaciones naturales</a>
<ul>
<li class="chapter" data-level="" data-path="pargen.html"><a href="pargen.html#estimaciones-de-heredabilidad-en-animales-domésticos"><i class="fa fa-check"></i>Estimaciones de heredabilidad en animales domésticos</a></li>
<li class="chapter" data-level="" data-path="pargen.html"><a href="pargen.html#estimaciones-de-heredabilidad-en-plantas"><i class="fa fa-check"></i>Estimaciones de heredabilidad en plantas</a></li>
<li class="chapter" data-level="" data-path="pargen.html"><a href="pargen.html#estimaciones-de-heredabilidad-en-condiciones-naturales"><i class="fa fa-check"></i>Estimaciones de heredabilidad en condiciones naturales</a></li>
</ul></li>
<li class="chapter" data-level="10.5" data-path="pargen.html"><a href="pargen.html#heredabilidad-y-filogenética"><i class="fa fa-check"></i><b>10.5</b> Heredabilidad y filogenética</a></li>
<li class="chapter" data-level="10.6" data-path="pargen.html"><a href="pargen.html#heredabilidad-en-la-era-genómica"><i class="fa fa-check"></i><b>10.6</b> Heredabilidad en la era genómica</a></li>
<li class="chapter" data-level="10.7" data-path="pargen.html"><a href="pargen.html#métodos-más-avanzados-de-estimación"><i class="fa fa-check"></i><b>10.7</b> Métodos más avanzados de estimación</a></li>
<li class="chapter" data-level="10.8" data-path="pargen.html"><a href="pargen.html#repetibilidad"><i class="fa fa-check"></i><b>10.8</b> Repetibilidad</a>
<ul>
<li class="chapter" data-level="" data-path="pargen.html"><a href="pargen.html#la-repetibilidad-como-cociente-de-varianzas"><i class="fa fa-check"></i>La repetibilidad como cociente de varianzas</a></li>
<li class="chapter" data-level="" data-path="pargen.html"><a href="pargen.html#la-repetibilidad-como-coeficiente-de-correlación-intraclase"><i class="fa fa-check"></i>La repetibilidad como coeficiente de correlación intraclase</a></li>
<li class="chapter" data-level="" data-path="pargen.html"><a href="pargen.html#la-relación-entre-la-heredabilidad-y-la-repetibilidad"><i class="fa fa-check"></i>La relación entre la heredabilidad y la repetibilidad</a></li>
<li class="chapter" data-level="" data-path="pargen.html"><a href="pargen.html#la-reducción-en-la-varianza-fenotípica-con-varias-medidas"><i class="fa fa-check"></i>La reducción en la varianza fenotípica con varias medidas</a></li>
</ul></li>
<li class="chapter" data-level="10.9" data-path="pargen.html"><a href="pargen.html#la-repetibilidad-como-herramienta-en-la-predicción"><i class="fa fa-check"></i><b>10.9</b> La repetibilidad como herramienta en la predicción</a></li>
<li class="chapter" data-level="10.10" data-path="pargen.html"><a href="pargen.html#evolucionabilidad"><i class="fa fa-check"></i><b>10.10</b> Evolucionabilidad</a></li>
<li class="chapter" data-level="" data-path="pargen.html"><a href="pargen.html#ejercicios-10"><i class="fa fa-check"></i>Ejercicios</a></li>
</ul></li>
<li class="chapter" data-level="11" data-path="selartifI.html"><a href="selartifI.html"><i class="fa fa-check"></i><b>11</b> Selección Artificial I</a>
<ul>
<li class="chapter" data-level="11.1" data-path="selartifI.html"><a href="selartifI.html#factores-de-corrección"><i class="fa fa-check"></i><b>11.1</b> Factores de corrección</a>
<ul>
<li class="chapter" data-level="" data-path="selartifI.html"><a href="selartifI.html#factores-de-corrección-aditivos"><i class="fa fa-check"></i>Factores de corrección aditivos</a></li>
<li class="chapter" data-level="" data-path="selartifI.html"><a href="selartifI.html#factores-de-corrección-multiplicativos"><i class="fa fa-check"></i>Factores de corrección multiplicativos</a></li>
<li class="chapter" data-level="" data-path="selartifI.html"><a href="selartifI.html#factores-de-corrección-lineales"><i class="fa fa-check"></i>Factores de corrección lineales</a></li>
</ul></li>
<li class="chapter" data-level="11.2" data-path="selartifI.html"><a href="selartifI.html#la-respuesta-a-la-selección-y-su-predicción"><i class="fa fa-check"></i><b>11.2</b> La respuesta a la selección y su predicción</a></li>
<li class="chapter" data-level="11.3" data-path="selartifI.html"><a href="selartifI.html#diferencial-de-selección-e-intensidad-de-selección"><i class="fa fa-check"></i><b>11.3</b> Diferencial de selección e intensidad de selección</a></li>
<li class="chapter" data-level="11.4" data-path="selartifI.html"><a href="selartifI.html#intensidad-de-selección-y-proporción-seleccionada"><i class="fa fa-check"></i><b>11.4</b> Intensidad de selección y proporción seleccionada</a>
<ul>
<li class="chapter" data-level="" data-path="selartifI.html"><a href="selartifI.html#de-dónde-sale-la-relación-ifraczp"><i class="fa fa-check"></i>¿De dónde sale la relación <span class="math inline">\(i=\frac{z}{p}\)</span>?</a></li>
<li class="chapter" data-level="" data-path="selartifI.html"><a href="selartifI.html#la-relación-entre-diferentes-formas-de-calcular-la-intensidad-de-selección"><i class="fa fa-check"></i>La relación entre diferentes formas de calcular la intensidad de selección</a></li>
</ul></li>
<li class="chapter" data-level="11.5" data-path="selartifI.html"><a href="selartifI.html#intervalo-generacional"><i class="fa fa-check"></i><b>11.5</b> Intervalo generacional</a></li>
<li class="chapter" data-level="11.6" data-path="selartifI.html"><a href="selartifI.html#medidas-de-la-respuesta"><i class="fa fa-check"></i><b>11.6</b> Medidas de la respuesta</a></li>
<li class="chapter" data-level="11.7" data-path="selartifI.html"><a href="selartifI.html#progreso-genético-generacional-y-anual"><i class="fa fa-check"></i><b>11.7</b> Progreso genético generacional y anual</a></li>
<li class="chapter" data-level="11.8" data-path="selartifI.html"><a href="selartifI.html#cambio-en-las-frecuencias-alélicas-bajo-selección-artificial"><i class="fa fa-check"></i><b>11.8</b> Cambio en las frecuencias alélicas bajo selección artificial</a></li>
<li class="chapter" data-level="11.9" data-path="selartifI.html"><a href="selartifI.html#el-diferencial-de-selección-direccional-y-la-identidad-de-robertson-price"><i class="fa fa-check"></i><b>11.9</b> El diferencial de selección direccional y la identidad de Robertson-Price</a></li>
<li class="chapter" data-level="" data-path="selartifI.html"><a href="selartifI.html#ejercicios-11"><i class="fa fa-check"></i>Ejercicios</a></li>
</ul></li>
<li class="chapter" data-level="12" data-path="correlacionada.html"><a href="correlacionada.html"><i class="fa fa-check"></i><b>12</b> Correlaciones y Respuesta Correlacionada</a>
<ul>
<li class="chapter" data-level="12.1" data-path="correlacionada.html"><a href="correlacionada.html#causas-genéticas-y-ambientales-de-las-correlaciones"><i class="fa fa-check"></i><b>12.1</b> Causas genéticas y ambientales de las correlaciones</a>
<ul>
<li class="chapter" data-level="" data-path="correlacionada.html"><a href="correlacionada.html#causas-de-las-correlaciones-genéticas"><i class="fa fa-check"></i>Causas de las correlaciones genéticas</a></li>
<li class="chapter" data-level="" data-path="correlacionada.html"><a href="correlacionada.html#causas-de-las-correlaciones-ambientales"><i class="fa fa-check"></i>Causas de las correlaciones ambientales</a></li>
</ul></li>
<li class="chapter" data-level="12.2" data-path="correlacionada.html"><a href="correlacionada.html#introducción-al-path-analysis"><i class="fa fa-check"></i><b>12.2</b> Introducción al “path analysis”</a>
<ul>
<li class="chapter" data-level="" data-path="correlacionada.html"><a href="correlacionada.html#la-relación-entre-las-correlaciones-fenotípica-genética-y-ambiental"><i class="fa fa-check"></i>La relación entre las correlaciones fenotípica, genética y ambiental</a></li>
</ul></li>
<li class="chapter" data-level="12.3" data-path="correlacionada.html"><a href="correlacionada.html#métodos-para-determinar-la-correlación-genética"><i class="fa fa-check"></i><b>12.3</b> Métodos para determinar la correlación genética</a></li>
<li class="chapter" data-level="12.4" data-path="correlacionada.html"><a href="correlacionada.html#la-correlación-fenotípica-y-su-relación-con-la-correlación-genética-aditiva-y-ambiental"><i class="fa fa-check"></i><b>12.4</b> La correlación fenotípica y su relación con la correlación genética aditiva y ambiental</a></li>
<li class="chapter" data-level="12.5" data-path="correlacionada.html"><a href="correlacionada.html#respuesta-correlacionada"><i class="fa fa-check"></i><b>12.5</b> Respuesta correlacionada</a>
<ul>
<li class="chapter" data-level="" data-path="correlacionada.html"><a href="correlacionada.html#eficiencia-en-relación-a-la-selección-directa"><i class="fa fa-check"></i>Eficiencia en relación a la selección directa</a></li>
<li class="chapter" data-level="" data-path="correlacionada.html"><a href="correlacionada.html#el-signo-de-la-correlación-y-el-beneficio-demérito-de-la-misma"><i class="fa fa-check"></i>El signo de la correlación y el beneficio-demérito de la misma</a></li>
</ul></li>
<li class="chapter" data-level="12.6" data-path="correlacionada.html"><a href="correlacionada.html#matrices-de-varianza-covarianza"><i class="fa fa-check"></i><b>12.6</b> Matrices de varianza-covarianza</a></li>
<li class="chapter" data-level="12.7" data-path="correlacionada.html"><a href="correlacionada.html#la-forma-generalizada-de-la-ecuación-del-criador"><i class="fa fa-check"></i><b>12.7</b> La forma generalizada de la ecuación del criador</a></li>
<li class="chapter" data-level="" data-path="correlacionada.html"><a href="correlacionada.html#ejercicios-12"><i class="fa fa-check"></i>Ejercicios</a></li>
</ul></li>
<li class="chapter" data-level="13" data-path="selartifII.html"><a href="selartifII.html"><i class="fa fa-check"></i><b>13</b> Selección Artificial II</a>
<ul>
<li class="chapter" data-level="13.1" data-path="selartifII.html"><a href="selartifII.html#criterios-y-objetivos-de-selección"><i class="fa fa-check"></i><b>13.1</b> Criterios y objetivos de selección</a></li>
<li class="chapter" data-level="13.2" data-path="selartifII.html"><a href="selartifII.html#selección-basada-en-un-solo-tipo-de-fuente-de-información"><i class="fa fa-check"></i><b>13.2</b> Selección basada en un solo tipo de fuente de información</a>
<ul>
<li class="chapter" data-level="" data-path="selartifII.html"><a href="selartifII.html#diferentes-tipos-de-selección-basados-en-estructuras-de-parentesco-definidas"><i class="fa fa-check"></i>Diferentes tipos de selección basados en estructuras de parentesco definidas</a></li>
<li class="chapter" data-level="" data-path="selartifII.html"><a href="selartifII.html#cálculo-de-las-heredabilidades-en-los-distintos-tipos-de-selección"><i class="fa fa-check"></i>Cálculo de las heredabilidades en los distintos tipos de selección</a></li>
</ul></li>
<li class="chapter" data-level="13.3" data-path="selartifII.html"><a href="selartifII.html#combinando-la-información-proporcionada-por-diferentes-tipos-de-parientes"><i class="fa fa-check"></i><b>13.3</b> Combinando la información proporcionada por diferentes tipos de parientes</a>
<ul>
<li class="chapter" data-level="" data-path="selartifII.html"><a href="selartifII.html#cálculo-de-los-elementos-de-la-matriz-de-varianzas-covarianzas-fenotípica"><i class="fa fa-check"></i>Cálculo de los elementos de la matriz de varianzas-covarianzas fenotípica</a></li>
<li class="chapter" data-level="" data-path="selartifII.html"><a href="selartifII.html#cálculo-de-los-elementos-del-vector-de-varianzas-covarianzas-genéticas"><i class="fa fa-check"></i>Cálculo de los elementos del vector de varianzas-covarianzas genéticas</a></li>
<li class="chapter" data-level="" data-path="selartifII.html"><a href="selartifII.html#información-propia-y-de-un-progenitor"><i class="fa fa-check"></i>Información propia y de un progenitor</a></li>
<li class="chapter" data-level="" data-path="selartifII.html"><a href="selartifII.html#información-propia-y-de-varios-hermanos-enteros"><i class="fa fa-check"></i>Información propia y de varios hermanos enteros</a></li>
<li class="chapter" data-level="" data-path="selartifII.html"><a href="selartifII.html#pruebas-de-progenie"><i class="fa fa-check"></i>Pruebas de progenie</a></li>
<li class="chapter" data-level="13.3.1" data-path="selartifII.html"><a href="selartifII.html#límites-de-la-eficiencia-en-pruebas-de-progenie"><i class="fa fa-check"></i><b>13.3.1</b> Límites de la eficiencia en pruebas de progenie</a></li>
<li class="chapter" data-level="" data-path="selartifII.html"><a href="selartifII.html#precisión-del-valor-de-cría-a-partir-del-índice"><i class="fa fa-check"></i>Precisión del valor de cría a partir del índice</a></li>
</ul></li>
<li class="chapter" data-level="13.4" data-path="selartifII.html"><a href="selartifII.html#métodos-de-selección-para-varias-características"><i class="fa fa-check"></i><b>13.4</b> Métodos de selección para varias características</a>
<ul>
<li class="chapter" data-level="" data-path="selartifII.html"><a href="selartifII.html#selección-en-tandem"><i class="fa fa-check"></i>Selección en tandem</a></li>
<li class="chapter" data-level="" data-path="selartifII.html"><a href="selartifII.html#niveles-independientes-de-rechazo"><i class="fa fa-check"></i>Niveles Independientes de Rechazo</a></li>
<li class="chapter" data-level="" data-path="selartifII.html"><a href="selartifII.html#índices-de-selección"><i class="fa fa-check"></i>Índices de Selección</a></li>
</ul></li>
<li class="chapter" data-level="13.5" data-path="selartifII.html"><a href="selartifII.html#índices-de-selección-para-múltiples-características"><i class="fa fa-check"></i><b>13.5</b> Índices de selección para múltiples características</a>
<ul>
<li class="chapter" data-level="" data-path="selartifII.html"><a href="selartifII.html#el-índice-de-smith-hazel"><i class="fa fa-check"></i>El índice de Smith-Hazel</a></li>
</ul></li>
<li class="chapter" data-level="13.6" data-path="selartifII.html"><a href="selartifII.html#métodos-y-técnicas-avanzadas-de-selección"><i class="fa fa-check"></i><b>13.6</b> Métodos y técnicas avanzadas de selección</a>
<ul>
<li class="chapter" data-level="" data-path="selartifII.html"><a href="selartifII.html#blup-las-ecuaciones-del-modelo-mixto-lineal"><i class="fa fa-check"></i>BLUP: las ecuaciones del modelo mixto lineal</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="selartifII.html"><a href="selartifII.html#ejercicios-13"><i class="fa fa-check"></i>Ejercicios</a></li>
</ul></li>
<li class="chapter" data-level="14" data-path="endoexo.html"><a href="endoexo.html"><i class="fa fa-check"></i><b>14</b> Endocría, exocría, consanguinidad y depresión endogámica</a>
<ul>
<li class="chapter" data-level="14.1" data-path="endoexo.html"><a href="endoexo.html#el-aumento-de-la-consanguinidad-a-partir-del-número-de-individuos"><i class="fa fa-check"></i><b>14.1</b> El aumento de la consanguinidad a partir del número de individuos</a>
<ul>
<li class="chapter" data-level="" data-path="endoexo.html"><a href="endoexo.html#qué-proporción-de-machos-es-necesaria-para-mantener-un-tamaño-efectivo-poblacional-mínimo"><i class="fa fa-check"></i>¿Qué proporción de machos es necesaria para mantener un tamaño efectivo poblacional mínimo?</a></li>
</ul></li>
<li class="chapter" data-level="14.2" data-path="endoexo.html"><a href="endoexo.html#el-coeficiente-de-consanguinidad-en-razas-lecheras"><i class="fa fa-check"></i><b>14.2</b> El coeficiente de consanguinidad en razas lecheras</a></li>
<li class="chapter" data-level="14.3" data-path="endoexo.html"><a href="endoexo.html#depresión-endogámica"><i class="fa fa-check"></i><b>14.3</b> Depresión endogámica</a>
<ul>
<li class="chapter" data-level="" data-path="endoexo.html"><a href="endoexo.html#los-riesgos-del-uso-masivo-de-reproductores-y-la-endogamia-elevada"><i class="fa fa-check"></i>Los riesgos del uso masivo de reproductores y la endogamia elevada</a></li>
<li class="chapter" data-level="" data-path="endoexo.html"><a href="endoexo.html#redistribución-en-la-varianza-genética"><i class="fa fa-check"></i>Redistribución en la varianza genética</a></li>
</ul></li>
<li class="chapter" data-level="14.4" data-path="endoexo.html"><a href="endoexo.html#exocría-y-heterosis"><i class="fa fa-check"></i><b>14.4</b> Exocría y Heterosis</a>
<ul>
<li class="chapter" data-level="" data-path="endoexo.html"><a href="endoexo.html#el-modelo-del-templo-griego"><i class="fa fa-check"></i>El modelo del templo griego</a></li>
<li class="chapter" data-level="" data-path="endoexo.html"><a href="endoexo.html#un-modelo-sencillo-de-heterosis-bajo-exocría"><i class="fa fa-check"></i>Un modelo sencillo de heterosis bajo exocría</a></li>
<li class="chapter" data-level="" data-path="endoexo.html"><a href="endoexo.html#complementariedad"><i class="fa fa-check"></i>Complementariedad</a></li>
<li class="chapter" data-level="" data-path="endoexo.html"><a href="endoexo.html#tipos-de-cruzamientos"><i class="fa fa-check"></i>Tipos de cruzamientos</a></li>
</ul></li>
<li class="chapter" data-level="14.5" data-path="endoexo.html"><a href="endoexo.html#modelo-genético-de-cruzamientos"><i class="fa fa-check"></i><b>14.5</b> Modelo genético de cruzamientos</a></li>
<li class="chapter" data-level="" data-path="endoexo.html"><a href="endoexo.html#ejercicios-14"><i class="fa fa-check"></i>Ejercicios</a></li>
<li class="chapter" data-level="" data-path="endoexo.html"><a href="endoexo.html#notas-al-pie"><i class="fa fa-check"></i>Notas al pie</a></li>
</ul></li>
<li class="chapter" data-level="15" data-path="GxE.html"><a href="GxE.html"><i class="fa fa-check"></i><b>15</b> Normas de reacción e interacción Genotipo x Ambiente</a>
<ul>
<li class="chapter" data-level="15.1" data-path="GxE.html"><a href="GxE.html#plasticidad-fenotípica-y-normas-de-reacción"><i class="fa fa-check"></i><b>15.1</b> Plasticidad fenotípica y normas de reacción</a></li>
<li class="chapter" data-level="15.2" data-path="GxE.html"><a href="GxE.html#interacción-gxe-en-dos-ambientes"><i class="fa fa-check"></i><b>15.2</b> Interacción GxE en dos ambientes</a></li>
<li class="chapter" data-level="15.3" data-path="GxE.html"><a href="GxE.html#correlación-genética-a-través-de-dos-ambientes"><i class="fa fa-check"></i><b>15.3</b> Correlación genética a través de dos ambientes</a>
<ul>
<li class="chapter" data-level="" data-path="GxE.html"><a href="GxE.html#procedimientos-de-estimación"><i class="fa fa-check"></i>Procedimientos de estimación</a></li>
</ul></li>
<li class="chapter" data-level="15.4" data-path="GxE.html"><a href="GxE.html#genética-cuantitativa-de-la-interacción-gxe"><i class="fa fa-check"></i><b>15.4</b> Genética cuantitativa de la interacción GxE</a></li>
<li class="chapter" data-level="15.5" data-path="GxE.html"><a href="GxE.html#otros-ejemplos-de-interacción-genotipo-ambiente"><i class="fa fa-check"></i><b>15.5</b> Otros ejemplos de interacción genotipo-ambiente</a></li>
<li class="chapter" data-level="" data-path="GxE.html"><a href="GxE.html#ejercicios-15"><i class="fa fa-check"></i>Ejercicios</a></li>
</ul></li>
<li class="chapter" data-level="16" data-path="apendiceuno.html"><a href="apendiceuno.html"><i class="fa fa-check"></i><b>16</b> APENDICE A: Conceptos Matemáticos Básicos</a></li>
<li class="chapter" data-level="17" data-path="apendicedos.html"><a href="apendicedos.html"><i class="fa fa-check"></i><b>17</b> APENDICE B: Cálculo Diferencial e Integral, ecuaciones diferenciales</a></li>
<li class="chapter" data-level="18" data-path="bibliografia.html"><a href="bibliografia.html"><i class="fa fa-check"></i><b>18</b> Bibliografía</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Introducción a la Genética de Poblaciones y a la Genética Cuantitativa</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="bibliografia" class="section level1 hasAnchor" number="18">
<h1><span class="header-section-number">Capítulo 18</span> Bibliografía<a href="bibliografia.html#bibliografia" class="anchor-section" aria-label="Anchor link to header"></a></h1>

<div id="refs" class="references csl-bib-body hanging-indent">
<div class="csl-entry">
2000. <a href="http://nitro.biosci.arizona.edu/zdownload/Volume2/Chapter23.pdf">http://nitro.biosci.arizona.edu/zdownload/Volume2/Chapter23.pdf</a>.
</div>
<div class="csl-entry">
———. 2021. <a href="https://queries.uscdcb.com/eval/summary/inbrd.cfm">https://queries.uscdcb.com/eval/summary/inbrd.cfm</a>.
</div>
<div class="csl-entry">
Abzhanov, A., W. P. Kuo, C. Hartmann, B. R. Grant, P. R. Grant, and C. J. Tabin. 2006. <span>“<span class="nocase"><span>T</span>he calmodulin pathway and evolution of elongated beak morphology in <span>D</span>arwin’s finches</span>.”</span> <em>Nature</em> 442 (7102): 563–67.
</div>
<div class="csl-entry">
Abzhanov, A., M. Protas, B. R. Grant, P. R. Grant, and C. J. Tabin. 2004. <span>“<span class="nocase"><span>B</span>mp4 and morphological variation of beaks in <span>D</span>arwin’s finches</span>.”</span> <em>Science</em> 305 (5689): 1462–65.
</div>
<div class="csl-entry">
Achtman, M. 1975. <span>“<span class="nocase"><span>M</span>ating aggregates in <span>E</span>scherichia coli conjugation</span>.”</span> <em>J Bacteriol</em> 123 (2): 505–15.
</div>
<div class="csl-entry">
Aïssani, B, and G Bernardi. 1991. <span>“CpG Islands, Genes and Isochores in the Genomes of Vertebrates.”</span> <em>Gene</em> 106 (2): 185–95. <a href="http://www.ncbi.nlm.nih.gov/pubmed/1937049">http://www.ncbi.nlm.nih.gov/pubmed/1937049</a>.
</div>
<div class="csl-entry">
Allais-Bonnet, A., C. Grohs, I. Medugorac, S. Krebs, A. Djari, A. Graf, S. Fritz, et al. 2013. <span>“<span class="nocase"><span>N</span>ovel insights into the bovine polled phenotype and horn ontogenesis in <span>B</span>ovidae</span>.”</span> <em>PLoS ONE</em> 8 (5): e63512.
</div>
<div class="csl-entry">
Aminov, R. I. 2011. <span>“<span class="nocase"><span>H</span>orizontal gene exchange in environmental microbiota</span>.”</span> <em>Front Microbiol</em> 2: 158.
</div>
<div class="csl-entry">
Arias, J. A., M. Keehan, P. Fisher, W. Coppieters, and R. Spelman. 2009. <span>“<span class="nocase"><span>A</span> high density linkage map of the bovine genome</span>.”</span> <em>BMC Genet</em> 10 (April): 18.
</div>
<div class="csl-entry">
Baudat, F., J. Buard, C. Grey, A. Fledel-Alon, C. Ober, M. Przeworski, G. Coop, and B. de Massy. 2010. <span>“<span class="nocase"><span>P</span><span>R</span><span>D</span><span>M</span>9 is a major determinant of meiotic recombination hotspots in humans and mice</span>.”</span> <em>Science</em> 327 (5967): 836–40.
</div>
<div class="csl-entry">
Bell, Washington, Jorge I. Urioste, Nelson Barlocco, Antonio Vadell, and Raquel Pérez Clariget. 2015. <span>“Genetic and Environmental Factors Affecting Reproductive Traits in Sows in an Outdoor Production System.”</span> <em>Livestock Science</em> 182: 101–7. https://doi.org/<a href="https://doi.org/10.1016/j.livsci.2015.10.025">https://doi.org/10.1016/j.livsci.2015.10.025</a>.
</div>
<div class="csl-entry">
Belle, Elise M S, Nick Smith, and Adam Eyre-Walker. 2002. <span>“Analysis of the Phylogenetic Distribution of Isochores in Vertebrates and a Test of the Thermal Stability Hypothesis.”</span> <em>Journal of Molecular Evolution</em> 55 (3): 356–63. <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=12187388">http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=12187388</a>.
</div>
<div class="csl-entry">
Bennett, G. L., R. G. Tait, S. D. Shackelford, T. L. Wheeler, D. A. King, E. Casas, and T. P. L. Smith. 2019. <span>“<span class="nocase"><span>E</span>nhanced estimates of carcass and meat quality effects for polymorphisms in myostatin and µ-calpain genes</span>.”</span> <em>J Anim Sci</em> 97 (2): 569–77.
</div>
<div class="csl-entry">
Benzecri, J P. 1980. <span>“Introduction a l’analyse Des Correspondances d’apres Un Exemple de Données Médicale.”</span> <em>Les Cahiers d’Analyse Des Données</em> V (3): 283–310.
</div>
<div class="csl-entry">
Bernardi, G, B Olofsson, J Filipski, M Zerial, J Salinas, G Cuny, M Meunier-Rotival, and F Rodier. 1985. <span>“The Mosaic Genome of Warm-Blooded Vertebrates.”</span> <em>Science</em> 228 (4702): 953–58. <a href="http://www.ncbi.nlm.nih.gov/pubmed/4001930">http://www.ncbi.nlm.nih.gov/pubmed/4001930</a>.
</div>
<div class="csl-entry">
Bezdı́ček, J., J. Šubrt, and R. Filipčı́k. 2008. <span>“The Effect of Inbreeding on Milk Traits in Holstein Cattle in the Czech Republic.”</span> <em>Archives Animal Breeding</em> 51 (5): 415–25. <a href="https://doi.org/10.5194/aab-51-415-2008">https://doi.org/10.5194/aab-51-415-2008</a>.
</div>
<div class="csl-entry">
Bigelow, Henry, and Burkhard Rost. 2009. <span>“Online Tools for Predicting Integral Membrane Proteins.”</span> Edited by Matthew J Peirce and RobinEditors Wait. <em>Methods In Molecular Biology Clifton Nj</em> 528: 3–23. <a href="http://www.springerlink.com/content/h869368601336862">http://www.springerlink.com/content/h869368601336862</a>.
</div>
<div class="csl-entry">
Blows, Mark, and Bruce Walsh. 2009. <span>“Spherical Cows Grazing in Flatland: Constraints to Selection and Adaptation.”</span> In <em>Adaptation and Fitness in Animal Populations: Evolutionary and Breeding Perspectives on Genetic Resource Management</em>, edited by Julius van der Werf, Hans-Ulrich Graser, Richard Frankham, and Cedric Gondro, 83–101. Dordrecht: Springer Netherlands. <a href="https://doi.org/10.1007/978-1-4020-9005-9_6">https://doi.org/10.1007/978-1-4020-9005-9_6</a>.
</div>
<div class="csl-entry">
Boggs, C. L. 2001. <span>“Species and Speciation,”</span> 14855–61. <a href="https://doi.org/10.1016/b0-08-043076-7/03073-4">https://doi.org/10.1016/b0-08-043076-7/03073-4</a>.
</div>
<div class="csl-entry">
Bourdieu, Pierre. 2002. <em>Le <span>B</span>al Des c<span>é</span>libataires. <span>C</span>rise de La Soci<span>é</span>t<span>é</span> Paysanne En <span>B</span><span>é</span>arn</em>. <span>É</span>ditions du Seuil.
</div>
<div class="csl-entry">
Bulmer, Michael George. 1980. <em>The Mathematical Theory of Quantitative Genetics</em>. Clarendon Press.
</div>
<div class="csl-entry">
Byars, S. G., D. Ewbank, D. R. Govindaraju, and S. C. Stearns. 2010. <span>“<span class="nocase"><span>C</span>olloquium papers: <span>N</span>atural selection in a contemporary human population</span>.”</span> <em>Proc Natl Acad Sci U S A</em> 107 Suppl 1 (January): 1787–92.
</div>
<div class="csl-entry">
Cannings, C., and A. W. Edwards. 1968. <span>“<span class="nocase"><span>N</span>atural selection and the de <span>F</span>inetti diagram</span>.”</span> <em>Ann. Hum. Genet.</em> 31 (4): 421–28.
</div>
<div class="csl-entry">
Cardellino, Ricardo, and Jaime Rovira. 1987. <em>Mejoramiento Gen<span>é</span>tico Animal</em>. 1era. ed. Editorial Hemisferio Sur.
</div>
<div class="csl-entry">
Castells Montes, Daniel. 2008. <span>“EVALUACI<span>Ó</span>n DE RESISTENCIA GEN<span>É</span>TICA DE OVINOS CORRIEDALE a LOS NEMATODOS GASTROINTESTINALES EN URUGUAY: Heredabilidad y Correlaciones Gen<span>é</span>ticas Entre El Recuento de Huevos de Nematodos y Caracter<span>ı́</span>sticas Productivas EVALUACI<span>Ó</span>n DE RESISTENCIA GEN<span>É</span>TICA DE OVINOS CORRIEDALE a LOS NEMATODOS GASTROINTESTINALES EN URUGUAY: Heredabilidad y Correlaciones Gen<span>é</span>ticas Entre El Recuento de Huevos de Nematodos y Caracter<span>ı́</span>sticas Productivas.”</span> Master’s thesis, Facultad de Agronom<span>ı́</span>a - Universidad de la Rep<span>ú</span>blica.
</div>
<div class="csl-entry">
Castle, W. E. 1903. <span>“<span class="nocase"><span>T</span>he laws of <span>G</span>alton and <span>M</span>endel and some laws governing race improvement</span>.”</span> <em>Proceedings of the American Academy of Arts and Sciences</em> 35 (8): 233–42.
</div>
<div class="csl-entry">
Cellon, C., R. R. Amadeu, J. W. Olmstead, M. R. Mattia, L. F. V. Ferrao, and P. R. Munoz. 2018. <span>“Estimation of Genetic Parameters and Prediction of Breeding Values in an Autotetraploid Blueberry Breeding Population with Extensive Pedigree Data.”</span> <em>Euphytica</em> 214 (87).
</div>
<div class="csl-entry">
Chan, K. M., Y. T. Liu, C. H. Ma, M. Jayaram, and S. Sau. 2013. <span>“<span class="nocase"><span>T</span>he 2 micron plasmid of <span>S</span>accharomyces cerevisiae: a miniaturized selfish genome with optimized functional competence</span>.”</span> <em>Plasmid</em> 70 (1): 2–17.
</div>
<div class="csl-entry">
Charlesworth, Brian, Russel Lande, and Montgomery Slatkin. 1982. <span>“A Neo-Darwinian Commentary on Macro Evolution.”</span> <em>Evolution</em> 36 (3): 474–98.
</div>
<div class="csl-entry">
Charlesworth, D., and J. H. Willis. 2009. <span>“<span class="nocase"><span>T</span>he genetics of inbreeding depression</span>.”</span> <em>Nat Rev Genet</em> 10 (11): 783–96.
</div>
<div class="csl-entry">
Ciappesoni, Gabriel, Ignacio De Barbieri, Diego Gimeno, and Fabio Montossi. 2011. <span>“Par<span>á</span>metros Gen<span>é</span>ticos Para <span class="nocase">ı́</span>Ndices de Selecci<span>ó</span>n de La Raza Merino Australiano.”</span> <em>Revista Argentina de Producci<span>ó</span>n Animal</em>.
</div>
<div class="csl-entry">
Cortazzo, Patricia, Carlos Cerveñansky, Mónica Marín, Claude Reiss, Ricardo Ehrlich, and Atilio Deana. 2002. <span>“Silent Mutations Affect in Vivo Protein Folding in Escherichia Coli.”</span> <em>Biochemical and Biophysical Research Communications</em> 293 (1): 537–41. <a href="http://www.ncbi.nlm.nih.gov/pubmed/12054634">http://www.ncbi.nlm.nih.gov/pubmed/12054634</a>.
</div>
<div class="csl-entry">
Cortes-Hernández, J., A. Garcı́a-Ruiz, C. G. Vásquez-Peláez, and F. J. Ruiz-Lopez. 2021. <span>“<span class="nocase"><span>C</span>orrelation of <span>G</span>enomic and <span>P</span>edigree <span>I</span>nbreeding <span>C</span>oefficients in <span>S</span>mall <span>C</span>attle <span>P</span>opulations</span>.”</span> <em>Animals (Basel)</em> 11 (11).
</div>
<div class="csl-entry">
Costantini, Maria, Rosalia Cammarano, and Giorgio Bernardi. 2009. <span>“The Evolution of Isochore Patterns in Vertebrate Genomes.”</span> <em>BMC Genomics</em> 10 (i): 146. <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2678159&amp;tool=pmcentrez&amp;rendertype=abstract">http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2678159&amp;tool=pmcentrez&amp;rendertype=abstract</a>.
</div>
<div class="csl-entry">
Cunningham, E. P. 1987. <span>“Crossbreeding the Greek Temple Model.”</span> <em>Journal of Animal Breeding and Genetics</em> 104 (1-2): 2–11.
</div>
<div class="csl-entry">
Darwin, Charles. 1876. <em>THE THE EFFECTS OF CROSS AND SELF FERTILISATION IN THE VEGETABLE KINGDOM OF CROSS AND SELF FERTILISATION IN THE VEGETABLE KINGDOM</em>. John Murray, Albemarle Street.
</div>
<div class="csl-entry">
Denny, Mark, and Steven Gaines. 2000. <em>Chance in Biology: Using Probability to Explore Nature</em>. Princeton University Press.
</div>
<div class="csl-entry">
Dickerson, Gordon E. 1973. <span>“Inbreeding and Heterosis in Animals.”</span> <em>Journal of Animal Science</em> Issue Symposium: 54–77.
</div>
<div class="csl-entry">
Dobzhansky, Theodosius. 1964. <span>“Biology, Molecular and Organismic.”</span> <em>American Zoologist</em> 4 (4): 443-452 ST - Biology, molecular and organismic. <a href="http://icb.oxfordjournals.org/cgi/reprint/4/4/443.pdf">http://icb.oxfordjournals.org/cgi/reprint/4/4/443.pdf</a>.
</div>
<div class="csl-entry">
Dohm, M. R. 2002. <span>“Repeatability Estimates Do Not Always Set an Upper Limit to Heritability.”</span> <em>Functional Ecology</em> 16: 273–80.
</div>
<div class="csl-entry">
Drake, J. W., B. Charlesworth, D. Charlesworth, and J. F. Crow. 1998. <span>“<span class="nocase"><span>R</span>ates of spontaneous mutation</span>.”</span> <em>Genetics</em> 148 (4): 1667–86.
</div>
<div class="csl-entry">
Drögemüller, C., T. Leeb, B. Harlizius, I. Tammen, O. Distl, M. Höltershinken, A. Gentile, A. Duchesne, and A. Eggen. 2007. <span>“<span class="nocase"><span>C</span>ongenital syndactyly in cattle: four novel mutations in the low density lipoprotein receptor-related protein 4 gene (<span>L</span><span>R</span><span>P</span>4)</span>.”</span> <em>BMC Genet</em> 8 (February): 5.
</div>
<div class="csl-entry">
Duret, L, D Mouchiroud, and C Gautier. 1995. <span>“Statistical Analysis of Vertebrate Sequences Reveals That Long Genes Are Scarce in GC-Rich Isochores.”</span> <em>Journal of Molecular Evolution</em> 40 (3): 308–17. <a href="http://www.springerlink.com/index/10.1007/BF00163235">http://www.springerlink.com/index/10.1007/BF00163235</a>.
</div>
<div class="csl-entry">
Duval, M., D. Dar, F. Carvalho, E. P. C. Rocha, R. Sorek, and P. Cossart. 2018. <span>“<span class="nocase"><span>H</span>fl<span>X</span>r, a homolog of a ribosome-splitting factor, mediates antibiotic resistance</span>.”</span> <em>Proc Natl Acad Sci U S A</em> 115 (52): 13359–64.
</div>
<div class="csl-entry">
Echeverrı́a, Pı́a, Ana Espasandin, Paola Gaiero, Ana Laura Sánchez, and Jorge I Urioste. 2012. <em>Manual de Pr<span>á</span>cticas Del Curso de Zootecnia</em>. Facultad de Agronom<span>ı́</span>a.
</div>
<div class="csl-entry">
Edwards, A. W. 1994. <span>“<span class="nocase"><span>T</span>he fundamental theorem of natural selection</span>.”</span> <em>Biol Rev Camb Philos Soc</em> 69 (4): 443–74.
</div>
<div class="csl-entry">
Ermolaeva, M D. 2001. <span>“Synonymous Codon Usage in Bacteria.”</span> <em>Current Issues in Molecular Biology</em> 3 (4): 91–97. <a href="http://www.ncbi.nlm.nih.gov/pubmed/11719972">http://www.ncbi.nlm.nih.gov/pubmed/11719972</a>.
</div>
<div class="csl-entry">
Espasandin, A. C., J. I. Urioste, H. Naya, and M. M. Alencar. 2013. <span>“Genotype×production Environment Interaction for Weaning Weight in Angus Populations of Brazil and Uruguay.”</span> <em>Livestock Science</em> 151 (2): 264–70. https://doi.org/<a href="https://doi.org/10.1016/j.livsci.2012.11.015">https://doi.org/10.1016/j.livsci.2012.11.015</a>.
</div>
<div class="csl-entry">
Eyre-Walker, A. 1992. <span>“Evidence That Both g + c Rich and g + c Poor Isochores Are Replicated Early and Late in the Cell Cycle.”</span> <em>Nucleic Acids Research</em> 20 (7): 1497–1501. <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=312229&amp;tool=pmcentrez&amp;rendertype=abstract">http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=312229&amp;tool=pmcentrez&amp;rendertype=abstract</a>.
</div>
<div class="csl-entry">
Eyre-Walker, A, and L D Hurst. 2001. <span>“The Evolution of Isochores.”</span> <em>Nature Reviews Genetics</em> 2 (7): 549–55. <a href="http://opus.bath.ac.uk/4362/">http://opus.bath.ac.uk/4362/</a>.
</div>
<div class="csl-entry">
Falconer, Douglas S. 1952. <span>“The Problem of Environment and Selection.”</span> <em>The American Naturalist</em> 86 (830): 293–98.
</div>
<div class="csl-entry">
———. 1960. <em>Introduction to Quantitative Genetics</em>. Oliver &amp; Boyd, Edinburgh.
</div>
<div class="csl-entry">
———. 1983. <em>Introducci<span>ó</span>n a La Gen<span>é</span>tica Cuantitativa</em>. CIA. Editorial Continental, S.A. de C.V.
</div>
<div class="csl-entry">
Falconer, Douglas S, and Trudy FC Mackay. 1996. <em>Quantitative Genetics</em>. Benjamin-Cummings Pub Co.
</div>
<div class="csl-entry">
Fang, L., Y. Li, Y. Zhang, D. Sun, L. Liu, Y. Zhang, and S. Zhang. 2013. <span>“<span class="nocase"><span>I</span>dentification of brachyspina syndrome carriers in <span>C</span>hinese <span>H</span>olstein cattle</span>.”</span> <em>J Vet Diagn Invest</em> 25 (4): 508–10.
</div>
<div class="csl-entry">
Felsenstein, Joseph. 2004. <em>Inferring Phylogenies</em>. Sinauer Associates, Inc.
</div>
<div class="csl-entry">
Finetti, B. de. 1926. <span>“<span class="nocase"><span>C</span>onsiderazioni matematiche sull’ereditarietà mendeliana</span>.”</span> <em>Metron</em> 6 (1): 3–41.
</div>
<div class="csl-entry">
Fisher, Ronald Aylmer. 1930. <em>The Genetical Theory of Natural Selection</em>. OXFORD UNIV PRESS.
</div>
<div class="csl-entry">
Fleischmann, R. D., M. D. Adams, O. White, R. A. Clayton, E. F. Kirkness, A. R. Kerlavage, C. J. Bult, J. F. Tomb, B. A. Dougherty, and J. M. Merrick. 1995. <span>“<span class="nocase"><span>W</span>hole-genome random sequencing and assembly of <span>H</span>aemophilus influenzae <span>R</span>d</span>.”</span> <em>Science</em> 269 (5223): 496–512.
</div>
<div class="csl-entry">
Fonseca, Dina M., Nusha Keyghobadi, Colin A. Malcolm, Ceylan Mehmet, Francis Schaffner, Motoyoshi Mogi, Robert C. Fleischer, and Richard C. Wilkerson. 2004. <span>“Emerging Vectors in the <span class="math inline">\(\less\)</span>i<span class="math inline">\(\greater\)</span>culex Pipiens<span class="math inline">\(\less\)</span>/i<span class="math inline">\(\greater\)</span> Complex.”</span> <em>Science</em> 303 (5663): 1535–38. <a href="https://doi.org/10.1126/science.1094247">https://doi.org/10.1126/science.1094247</a>.
</div>
<div class="csl-entry">
Fox, Jesse M, and Ivan Erill. 2010. <span>“Relative Codon Adaptation: A Generic Codon Bias Index for Prediction of Gene Expression.”</span> <em>DNA Research an International Journal for Rapid Publication of Reports on Genes and Genomes</em> 17 (3): 185–96. <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2885275&amp;tool=pmcentrez&amp;rendertype=abstract">http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2885275&amp;tool=pmcentrez&amp;rendertype=abstract</a>.
</div>
<div class="csl-entry">
Frioni, Nicolás. 2012. <span>“Estimaci<span>ó</span>n de Heredabilidad Del Intervalo Entre Partos y Su Correlaci<span>ó</span>n Gen<span>é</span>tica Con Producci<span>ó</span>n de Leche En Ganado Holando Uruguayo.”</span> Facultad de Agronom<span>ı́</span>a - Universidad de la Rep<span>ú</span>blica.
</div>
<div class="csl-entry">
———. 2016. <span>“Estimaci<span>ó</span>n de Par<span>á</span>metros Gen<span>é</span>ticos En Caracter<span>ı́</span>sticas Reproductivas y Productivas Para La Raza Holando En Uruguay.”</span> Master’s thesis, Facultad de Agronom<span>ı́</span>a - Universidad de la Rep<span>ú</span>blica.
</div>
<div class="csl-entry">
Galton, Francis. 1886. <span>“Regression Towards Mediocrity in Hereditary Stature.”</span> <em>Journal of the Anthropological Institute</em> 15: 246–63.
</div>
<div class="csl-entry">
Garrow, Andrew G, Alison Agnew, and David R Westhead. 2005. <span>“TMB-Hunt: An Amino Acid Composition Based Method to Screen Proteomes for Beta-Barrel Transmembrane Proteins.”</span> <em>BMC Bioinformatics</em> 6 (1): 56. <a href="http://www.ncbi.nlm.nih.gov/pubmed/15769290">http://www.ncbi.nlm.nih.gov/pubmed/15769290</a>.
</div>
<div class="csl-entry">
Gianola, Daniel. 1982. <span>“Theory and Analysis of Threshold Characters.”</span> <em>Journal of Animal Science</em> 54 (5): 1079–96.
</div>
<div class="csl-entry">
Gianola, D., S. Qanbari, and H. Simianer. 2013. <span>“<span class="nocase"><span>A</span>n evaluation of a novel estimator of linkage disequilibrium</span>.”</span> <em>Heredity (Edinb)</em> 111 (4): 275–85.
</div>
<div class="csl-entry">
Gillespie, J. H. 2000. <span>“<span class="nocase"><span>G</span>enetic drift in an infinite population. <span>T</span>he pseudohitchhiking model</span>.”</span> <em>Genetics</em> 155 (2): 909–19.
</div>
<div class="csl-entry">
Gillespie, John H. 2004. <em>Population Genetics. A Concise Guide</em>. The Johns Hopkins University Press.
</div>
<div class="csl-entry">
Goldberg, Virginia, Gabriel Ciappesoni, and Ignacio Aguilar. 2012. <span>“Genetic Parameters for Nematode Resistance in Periparturient Ewes and Post-Weaning Lambs in Uruguayan Merino Sheep.”</span> <em>Livestock Science</em> 147 (1): 181–87. https://doi.org/<a href="https://doi.org/10.1016/j.livsci.2012.05.003">https://doi.org/10.1016/j.livsci.2012.05.003</a>.
</div>
<div class="csl-entry">
González, Alejandra V., and Ernesto Gianoli. 2004. <span>“Morphological Plasticity in Response to Shading in Three Convolvulus Species of Different Ecological Breadth.”</span> <em>Acta Oecologica</em> 26 (June): 185–90.
</div>
<div class="csl-entry">
Grandi, M. Florencia, Larissa R. de Oliveira, Silvana L. Dans, and Enrique A. Crespo. 2012. <span>“A Hunted Population in Recovery: Effective Population Size for South American Sea Lions from Patagonia.”</span> <em>Animal Biology</em> 62 (4): 433–50. https://doi.org/<a href="https://doi.org/10.1163/157075612X642941">https://doi.org/10.1163/157075612X642941</a>.
</div>
<div class="csl-entry">
Grigoriev, A. 1998. <span>“Analyzing Genomes with Cumulative Skew Diagrams.”</span> <em>Nucleic Acids Research</em> 26 (10): 2286–90. <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=147580&amp;tool=pmcentrez&amp;rendertype=abstract">http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=147580&amp;tool=pmcentrez&amp;rendertype=abstract</a>.
</div>
<div class="csl-entry">
Gupta, A. P., and R. C. Lewontin. 1982. <span>“<span><span>A</span> <span>S</span><span>T</span><span>U</span><span>D</span><span>Y</span> <span>O</span><span>F</span> <span>R</span><span>E</span><span>A</span><span>C</span><span>T</span><span>I</span><span>O</span><span>N</span> <span>N</span><span>O</span><span>R</span><span>M</span><span>S</span> <span>I</span><span>N</span> <span>N</span><span>A</span><span>T</span><span>U</span><span>R</span><span>A</span><span>L</span> <span>P</span><span>O</span><span>P</span><span>U</span><span>L</span><span>A</span><span>T</span><span>I</span><span>O</span><span>N</span><span>S</span> <span>O</span><span>F</span> <span>D</span><span>R</span><span>O</span><span>S</span><span>O</span><span>P</span><span>H</span><span>I</span><span>L</span><span>A</span> <span>P</span><span>S</span><span>E</span><span>U</span><span>D</span><span>O</span><span>O</span><span>B</span><span>S</span><span>C</span><span>U</span><span>R</span><span>A</span></span>.”</span> <em>Evolution</em> 36 (5): 934–48.
</div>
<div class="csl-entry">
Hadfield, Jarrod D. 2010. <span>“MCMC Methods for Multi-Response Generalized Linear Mixed Models: The <span>MCMCglmm</span> <span>R</span> Package.”</span> <em>Journal of Statistical Software</em> 33 (2): 1–22. <a href="https://www.jstatsoft.org/v33/i02/">https://www.jstatsoft.org/v33/i02/</a>.
</div>
<div class="csl-entry">
Hamilton, Matthew B. 2009. <em>Population Genetics</em>. Wiley-Blackwell.
</div>
<div class="csl-entry">
Handelsman, J., M. R. Rondon, S. F. Brady, J. Clardy, and R. M. Goodman. 1998. <span>“<span class="nocase"><span>M</span>olecular biological access to the chemistry of unknown soil microbes: a new frontier for natural products</span>.”</span> <em>Chem Biol</em> 5 (10): R245–249.
</div>
<div class="csl-entry">
Hardy, G. H. 1908. <span>“<span><span>M</span><span>E</span><span>N</span><span>D</span><span>E</span><span>L</span><span>I</span><span>A</span><span>N</span> <span>P</span><span>R</span><span>O</span><span>P</span><span>O</span><span>R</span><span>T</span><span>I</span><span>O</span><span>N</span><span>S</span> <span>I</span><span>N</span> <span>A</span> <span>M</span><span>I</span><span>X</span><span>E</span><span>D</span> <span>P</span><span>O</span><span>P</span><span>U</span><span>L</span><span>A</span><span>T</span><span>I</span><span>O</span><span>N</span></span>.”</span> <em>Science</em> 28 (706): 49–50.
</div>
<div class="csl-entry">
Hartl, Daniel L. 2020. <em>A Primer of Population Genetics and Genomics</em>. 4th ed. London, England: Oxford University Press.
</div>
<div class="csl-entry">
Hartl, Daniel L, and Andrew G Clark. 2007. <em>Principles of Population Genetics</em>. 4th. ed. Sinauer.
</div>
<div class="csl-entry">
Havenstein, G. B., P. R. Ferket, S. E. Scheideler, and B. T. Larson. 1994. <span>“<span class="nocase"><span>G</span>rowth, livability, and feed conversion of 1957 vs 1991 broilers when fed "typical" 1957 and 1991 broiler diets</span>.”</span> <em>Poult Sci</em> 73 (12): 1785–94.
</div>
<div class="csl-entry">
Henderson, C. R., and R. L. Quaas. 1976. <span>“<span>Multiple Trait Evaluation Using Relatives’ Records</span>.”</span> <em>Journal of Animal Science</em> 43 (6): 1188–97. <a href="https://doi.org/10.2527/jas1976.4361188x">https://doi.org/10.2527/jas1976.4361188x</a>.
</div>
<div class="csl-entry">
Henderson, Charles Roy. 1949. <span>“Estimation of Changes in Herd Environment.”</span> <em>Journal of Dairy Science</em> 32: 706.
</div>
<div class="csl-entry">
———. 1950. <span>“Estimation of Genetic Parameters.”</span> <em>Ann Math Stat</em> 21: 309–10.
</div>
<div class="csl-entry">
———. 1959. <span>“The Estimation of Environmental and Genetic Trends from Records Subject to Culling.”</span> <em>Biometrics</em> 15 (2): 192–218.
</div>
<div class="csl-entry">
Hill, W. G. 1972. <span>“<span class="nocase"><span>E</span>stimation of realised heritabilities from selection experiments. <span>I</span>. <span>D</span>ivergent selection</span>.”</span> <em>Biometrics</em> 28 (3): 747–65.
</div>
<div class="csl-entry">
Holsinger, K. E., and B. S. Weir. 2009. <span>“<span class="nocase"><span>G</span>enetics in geographically structured populations: defining, estimating and interpreting <span>F</span>(<span>S</span><span>T</span>)</span>.”</span> <em>Nat Rev Genet</em> 10 (9): 639–50.
</div>
<div class="csl-entry">
Hoque, M. N., A. Istiaq, R. A. Clement, K. M. Gibson, O. Saha, O. K. Islam, R. A. Abir, et al. 2020. <span>“<span class="nocase"><span>I</span>nsights <span>I</span>nto the <span>R</span>esistome of <span>B</span>ovine <span>C</span>linical <span>M</span>astitis <span>M</span>icrobiome, a <span>K</span>ey <span>F</span>actor in <span>D</span>isease <span>C</span>omplication</span>.”</span> <em>Front Microbiol</em> 11: 860.
</div>
<div class="csl-entry">
Houchmandzadeh, B. 2014. <span>“<span>A</span>n Alternative to the Breeder’s and <span>L</span>ande’s Equations.”</span> <em>G3 (Bethesda)</em> 4 (1): 97–108.
</div>
<div class="csl-entry">
Houle, D. 1992. <span>“<span class="nocase"><span>C</span>omparing evolvability and variability of quantitative traits</span>.”</span> <em>Genetics</em> 130 (1): 195–204.
</div>
<div class="csl-entry">
Iraola, G., L. Spangenberg, B. Lopes Bastos, M. Graña, L. Vasconcelos, Á. Almeida, G. Greif, C. Robello, P. Ristow, and H. Naya. 2016. <span>“<span class="nocase"><span>T</span>ranscriptome <span>S</span>equencing <span>R</span>eveals <span>W</span>ide <span>E</span>xpression <span>R</span>eprogramming of <span>B</span>asal and <span>U</span>nknown <span>G</span>enes in <span>L</span>eptospira biflexa <span>B</span>iofilms</span>.”</span> <em>mSphere</em> 1 (2).
</div>
<div class="csl-entry">
Johannsen, Wilhelm L. 1903. <em><span>Ü</span>ber Erblichkeit in Populationen Und in Reinen Linien : Ein Beitrag Zur Beleuchtung Schwebender Selektionsfragen</em>. Fischer (Jena).
</div>
<div class="csl-entry">
Jong, G. de. 1990. <span>“Quantitative Genetics of Reaction Norms.”</span> <em>J. Evol. Biol</em>, 447–68.
</div>
<div class="csl-entry">
Kagawa, Y, H Nojima, N Nukiwa, M Ishizuka, T Nakajima, T Yasuhara, T Tanaka, and T Oshima. 1984. <span>“High Guanine Plus Cytosine Content in the Third Letter of Codons of an Extreme Thermophile. DNA Sequence of the Isopropylmalate Dehydrogenase of Thermus Thermophilus.”</span> <em>The Journal of Biological Chemistry</em> 259 (5): 2956–60.
</div>
<div class="csl-entry">
Kawashima, S, H Ogata, and M Kanehisa. 2000. <span>“AAindex: Amino Acid Index Database.”</span> <em>Nucleic Acids Research</em> 27 (1): 374. <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2238890&amp;tool=pmcentrez&amp;rendertype=abstract">http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2238890&amp;tool=pmcentrez&amp;rendertype=abstract</a>.
</div>
<div class="csl-entry">
Kazanskaya, O V, E A Severtzova, K A Barth, G V Ermakova, S A Lukyanov, A O Benyumov, M Pannese, E Boncinelli, S W Wilson, and A G Zaraisky. 1997. <span>“Methylation Patterns in the Isochores of Vertebrate Genomes.”</span> <em>Gene</em> 205 (1-2): 25–34. <a href="http://www.ncbi.nlm.nih.gov/pubmed/9461385">http://www.ncbi.nlm.nih.gov/pubmed/9461385</a>.
</div>
<div class="csl-entry">
Kellenberger, Roman T., Kelsey J. R. P. Byers, Rita M. De Brito Francisco, Yannick M. Staedler, Amy M. LaFountain, Jürg Schönenberger, Florian P. Schiestl, and Philipp M. Schlüter. 2019. <span>“Emergence of a Floral Colour Polymorphism by Pollinator-Mediated Overdominance.”</span> <em>Nature Communications</em> 10 (1). <a href="https://doi.org/10.1038/s41467-018-07936-x">https://doi.org/10.1038/s41467-018-07936-x</a>.
</div>
<div class="csl-entry">
Keller, Lukas F., and Donald M. Waller. 2002. <span>“Inbreeding Effects in Wild Populations.”</span> <em>Trends in Ecology and Evolution</em> 17 (5): 230–41.
</div>
<div class="csl-entry">
Kimura, M. 1962. <span>“<span class="nocase"><span>O</span>n the probability of fixation of mutant genes in a population</span>.”</span> <em>Genetics</em> 47 (June): 713–19.
</div>
<div class="csl-entry">
Kimura, M., and T. Ohta. 1969. <span>“<span class="nocase"><span>T</span>he <span>A</span>verage <span>N</span>umber of <span>G</span>enerations until <span>F</span>ixation of a <span>M</span>utant <span>G</span>ene in a <span>F</span>inite <span>P</span>opulation</span>.”</span> <em>Genetics</em> 61 (3): 763–71.
</div>
<div class="csl-entry">
Kingman, JFC. 1982a. <span>“On the Genealogy of Large Populations.”</span> <em>Journal of Applied Probability</em> 19A: 27–43.
</div>
<div class="csl-entry">
———. 1982b. <span>“The Coalescent.”</span> <em>Stochastic Processes and Their Applications</em> 13: 235–48.
</div>
<div class="csl-entry">
Kipp, S., D. Segelke, S. Schierenbeck, F. Reinhardt, R. Reents, C. Wurmser, H. Pausch, et al. 2016. <span>“<span class="nocase"><span>I</span>dentification of a haplotype associated with cholesterol deficiency and increased juvenile mortality in <span>H</span>olstein cattle</span>.”</span> <em>J Dairy Sci</em> 99 (11): 8915–31.
</div>
<div class="csl-entry">
Kong, A., D. F. Gudbjartsson, J. Sainz, G. M. Jonsdottir, S. A. Gudjonsson, B. Richardsson, S. Sigurdardottir, et al. 2002. <span>“<span class="nocase"><span>A</span> high-resolution recombination map of the human genome</span>.”</span> <em>Nat Genet</em> 31 (3): 241–47.
</div>
<div class="csl-entry">
Kong, A., G. Thorleifsson, D. F. Gudbjartsson, G. Masson, A. Sigurdsson, A. Jonasdottir, G. B. Walters, et al. 2010. <span>“<span class="nocase"><span>F</span>ine-scale recombination rate differences between sexes, populations and individuals</span>.”</span> <em>Nature</em> 467 (7319): 1099–1103.
</div>
<div class="csl-entry">
Lande, R. 1979. <span>“<span><span>Q</span><span>U</span><span>A</span><span>N</span><span>T</span><span>I</span><span>T</span><span>A</span><span>T</span><span>I</span><span>V</span><span>E</span> <span>G</span><span>E</span><span>N</span><span>E</span><span>T</span><span>I</span><span>C</span> <span>A</span><span>N</span><span>A</span><span>L</span><span>Y</span><span>S</span><span>I</span><span>S</span> <span>O</span><span>F</span> <span>M</span><span>U</span><span>L</span><span>T</span><span>I</span><span>V</span><span>A</span><span>R</span><span>I</span><span>A</span><span>T</span><span>E</span> <span>E</span><span>V</span><span>O</span><span>L</span><span>U</span><span>T</span><span>I</span><span>O</span><span>N</span>, <span>A</span><span>P</span><span>P</span><span>L</span><span>I</span><span>E</span><span>D</span> <span>T</span><span>O</span> <span>B</span><span>R</span><span>A</span><span>I</span><span>N</span>:<span>B</span><span>O</span><span>D</span><span>Y</span> <span>S</span><span>I</span><span>Z</span><span>E</span> <span>A</span><span>L</span><span>L</span><span>O</span><span>M</span><span>E</span><span>T</span><span>R</span><span>Y</span></span>.”</span> <em>Evolution</em> 33 (1Part2): 402–16.
</div>
<div class="csl-entry">
Lane, D. J., B. Pace, G. J. Olsen, D. A. Stahl, M. L. Sogin, and N. R. Pace. 1985. <span>“<span class="nocase"><span>R</span>apid determination of 16<span>S</span> ribosomal <span>R</span><span>N</span><span>A</span> sequences for phylogenetic analyses</span>.”</span> <em>Proc Natl Acad Sci U S A</em> 82 (20): 6955–59.
</div>
<div class="csl-entry">
Li, Ching Chun. 1955. <em>Population Genetics</em>. First. Chicago Press.
</div>
<div class="csl-entry">
Loáisiga, C. H., O. Rocha, A. K. Brantestam, B. Salomon, and A. Merker. 2012. <span>“Genetic Diversity and Gene Flow in Six Accessions of Meso-America Teosintes.”</span> <em>Genet Resour Crop Evol</em> 59: 95–111.
</div>
<div class="csl-entry">
Lobo, Ingrid. 2008. <span>“Pleiotropy: One Gene Can Affect Multiple Traits.”</span> <em>Nature Education</em> 1 (1): 10.
</div>
<div class="csl-entry">
Loehlin, John C, and A. Alexander Beaujean. 2017. <em>Latent Variable Models: An Introduction to Factor, Path, and Structural Equation Analysis</em>. Fifth. Routledge (Taylor; Francis group).
</div>
<div class="csl-entry">
López-Correa, Rodrigo D. 2013. <span>“Variabilidad Gen<span>é</span>tica Del Desempe<span>ñ</span>o de Caballos Criollos En Pruebas de Resistencia.”</span> Master’s thesis, Facultad de Agronom<span>ı́</span>a - Universidad de la Rep<span>ú</span>blica.
</div>
<div class="csl-entry">
López-Correa, Rodrigo D, Francisco Peñagaricano, Gabriel Rovere, and Jorge Ignacio Urioste. 2018. <span>“Genetic Parameter Estimation for Long Endurance Trials in the Uruguayan Criollo Horse.”</span> <em>Journal of Animal Breeding</em> 135 (3): 186–93.
</div>
<div class="csl-entry">
Lstibůrek, M., V. Bittner, G. R. Hodge, J. Picek, and T. F. C. Mackay. 2018. <span>“<span class="nocase"><span>E</span>stimating <span>R</span>ealized <span>H</span>eritability in <span>P</span>anmictic <span>P</span>opulations</span>.”</span> <em>Genetics</em> 208 (1): 89–95.
</div>
<div class="csl-entry">
Lynch, M. 1991. <span>“<span><span>M</span><span>E</span><span>T</span><span>H</span><span>O</span><span>D</span><span>S</span> <span>F</span><span>O</span><span>R</span> <span>T</span><span>H</span><span>E</span> <span>A</span><span>N</span><span>A</span><span>L</span><span>Y</span><span>S</span><span>I</span><span>S</span> <span>O</span><span>F</span> <span>C</span><span>O</span><span>M</span><span>P</span><span>A</span><span>R</span><span>A</span><span>T</span><span>I</span><span>V</span><span>E</span> <span>D</span><span>A</span><span>T</span><span>A</span> <span>I</span><span>N</span> <span>E</span><span>V</span><span>O</span><span>L</span><span>U</span><span>T</span><span>I</span><span>O</span><span>N</span><span>A</span><span>R</span><span>Y</span> <span>B</span><span>I</span><span>O</span><span>L</span><span>O</span><span>G</span><span>Y</span></span>.”</span> <em>Evolution</em> 45 (5): 1065–80.
</div>
<div class="csl-entry">
Lynch, Michael, and Bruce Walsh. 1998. <em>Genetics and Analysis of Quantitative Traits</em>. Sinauer Associates (Oxford University Press).
</div>
<div class="csl-entry">
Ma, Li, Tad S. Sonstegard, John B. Cole, Curtis P. VanTassell, George R. Wiggans, Brian A. Crooker, Cheng Tan, Dzianis Prakapenka, George E. Liu, and Yang Da. 2019. <span>“Genome Changes Due to Artificial Selection in u.s. Holstein Cattle.”</span> <em><span>BMC</span> Genomics</em> 20 (1). <a href="https://doi.org/10.1186/s12864-019-5459-x">https://doi.org/10.1186/s12864-019-5459-x</a>.
</div>
<div class="csl-entry">
Marco de Lima, B., E. P. Cappa, O. B. Silva-Junior, C. Garcia, S. D. Mansfield, and D. Grattapaglia. 2019. <span>“<span class="nocase"><span>Q</span>uantitative genetic parameters for growth and wood properties in <span>E</span>ucalyptus "urograndis" hybrid using near-infrared phenotyping and genome-wide <span>S</span><span>N</span><span>P</span>-based relationships</span>.”</span> <em>PLoS One</em> 14 (6): e0218747.
</div>
<div class="csl-entry">
Maynard Smith, J., and J. Haigh. 1974. <span>“<span class="nocase"><span>T</span>he hitch-hiking effect of a favourable gene</span>.”</span> <em>Genet Res</em> 23 (1): 23–35.
</div>
<div class="csl-entry">
Mc Parland, S., J. F. Kearney, M. Rath, and D. P. Berry. 2007. <span>“<span class="nocase"><span>I</span>nbreeding effects on milk production, calving performance, fertility, and conformation in <span>I</span>rish <span>H</span>olstein-<span>F</span>riesians</span>.”</span> <em>J Dairy Sci</em> 90 (9): 4411–19.
</div>
<div class="csl-entry">
McEwan, C E, D Gatherer, and N R McEwan. 1998. <span>“Nitrogen-Fixing Aerobic Bacteria Have Higher Genomic GC Content Than Non-Fixing Species Within the Same Genus.”</span> <em>Hereditas</em> 128 (2): 173–78. <a href="http://www.ncbi.nlm.nih.gov/pubmed/9687237">http://www.ncbi.nlm.nih.gov/pubmed/9687237</a>.
</div>
<div class="csl-entry">
Medugorac, I., D. Seichter, A. Graf, I. Russ, H. Blum, K. H. Göpel, S. Rothammer, M. Förster, and S. Krebs. 2012. <span>“<span class="nocase"><span>B</span>ovine polledness–an autosomal dominant trait with allelic heterogeneity</span>.”</span> <em>PLoS ONE</em> 7 (6): e39477.
</div>
<div class="csl-entry">
Miglior, F., B. Szkotnicki, and E. B. Burnside. 1992. <span>“<span class="nocase"><span>A</span>nalysis of levels of inbreeding and inbreeding depression in <span>J</span>ersey cattle</span>.”</span> <em>J Dairy Sci</em> 75 (4): 1112–18.
</div>
<div class="csl-entry">
Moldovan, M. A. 2019. <span>“<span class="nocase">[<span>P</span>rokaryotic and <span>M</span>itochondrial <span>L</span>inear <span>G</span>enomes: <span>T</span>heir <span>G</span>enesis, <span>E</span>volutionary <span>S</span>ignificance, and the <span>P</span>roblem of <span>R</span>eplicating <span>C</span>hromosome <span>E</span>nds]</span>.”</span> <em>Mol Biol (Mosk)</em> 53 (2): 218–24.
</div>
<div class="csl-entry">
Mrode, Raphael. 2014. <em>Linear Models for the Prediction of Animal Breeding Values</em>. 3rd ed. CABI.
</div>
<div class="csl-entry">
Musto, Héctor, Hugo Naya, Alejandro Zavala, Héctor Romero, Fernando Alvarez-Valín, and Giorgio Bernardi. 2004. <span>“Correlations Between Genomic GC Levels and Optimal Growth Temperatures in Prokaryotes.”</span> <em>FEBS Letters</em> 573 (1-3): 73–77. <a href="http://www.ncbi.nlm.nih.gov/pubmed/15327978">http://www.ncbi.nlm.nih.gov/pubmed/15327978</a>.
</div>
<div class="csl-entry">
Nagahata, H. 2004. <span>“<span class="nocase"><span>B</span>ovine leukocyte adhesion deficiency (<span>B</span><span>L</span><span>A</span><span>D</span>): a review</span>.”</span> <em>J Vet Med Sci</em> 66 (12): 1475–82.
</div>
<div class="csl-entry">
Naya, H., D. Gianola, H. Romero, J. I. Urioste, and H. Musto. 2006. <span>“<span class="nocase"><span>I</span>nferring parameters shaping amino acid usage in prokaryotic genomes via <span>B</span>ayesian <span>M</span><span>C</span><span>M</span><span>C</span> methods</span>.”</span> <em>Mol Biol Evol</em> 23 (1): 203–11.
</div>
<div class="csl-entry">
Naya, H., F. Peñagaricano, and J. I. Urioste. 2017. <span>“<span class="nocase"><span>M</span>odelling female fertility traits in beef cattle using linear and non-linear models</span>.”</span> <em>J Anim Breed Genet</em> 134 (3): 202–12.
</div>
<div class="csl-entry">
Naya, Hugo, Héctor Romero, Alejandro Zavala, Beatriz Alvarez, and Héctor Musto. 2002. <span>“Aerobiosis Increases the Genomic Guanine Plus Cytosine Content (GC%) in Prokaryotes.”</span> <em>Journal of Molecular Evolution</em> 55 (3): 260–64. <a href="http://www.ncbi.nlm.nih.gov/pubmed/12187379">http://www.ncbi.nlm.nih.gov/pubmed/12187379</a>.
</div>
<div class="csl-entry">
Naya, H., Jorge I Urioste, and J. Franco. 2002. <em>Identification of Production Environments and Presence of GxE Interactions in Uruguay Using Holstein Herds Records.</em> Vol. 2002. Session 1.
</div>
<div class="csl-entry">
Nonnemann, B., U. Lyhs, L. Svennesen, K. A. Kristensen, I. C. Klaas, and K. Pedersen. 2019. <span>“<span class="nocase"><span>B</span>ovine mastitis bacteria resolved by <span>M</span><span>A</span><span>L</span><span>D</span><span>I</span>-<span>T</span><span>O</span><span>F</span> mass spectrometry</span>.”</span> <em>J Dairy Sci</em> 102 (3): 2515–24.
</div>
<div class="csl-entry">
Nordborg, M., J. O. Borevitz, J. Bergelson, C. C. Berry, J. Chory, J. Hagenblad, M. Kreitman, et al. 2002. <span>“<span class="nocase"><span>T</span>he extent of linkage disequilibrium in <span>A</span>rabidopsis thaliana</span>.”</span> <em>Nat Genet</em> 30 (2): 190–93.
</div>
<div class="csl-entry">
Novembre, John A. 2002. <span>“Accounting for Background Nucleotide Composition When Measuring Codon Usage Bias.”</span> <em>Molecular Biology and Evolution</em> 19 (8): 1390–94. <a href="http://mbe.oxfordjournals.org/content/19/8/1390.long">http://mbe.oxfordjournals.org/content/19/8/1390.long</a>.
</div>
<div class="csl-entry">
Novozhilov, Artem S, Yuri I Wolf, and Eugene V Koonin. 2007. <span>“Evolution of the Genetic Code: Partial Optimization of a Random Code for Robustness to Translation Error in a Rugged Fitness Landscape.”</span> <em>Biology Direct</em> 2 (1): 24. <a href="http://www.ncbi.nlm.nih.gov/pubmed/17956616">http://www.ncbi.nlm.nih.gov/pubmed/17956616</a>.
</div>
<div class="csl-entry">
Otto, Sarah P., and Troy Day. 2007. <em>A Biologist’s Guide to Mathematical Modeling in Ecology and Evolution</em>. Princeton University Press.
</div>
<div class="csl-entry">
Pielou, E. C. 1969. <em>An Introduction to Mathematical Ecology</em>. Wiley-Interscience.
</div>
<div class="csl-entry">
Price, G. R. 1970. <span>“<span class="nocase"><span>S</span>election and covariance</span>.”</span> <em>Nature</em> 227 (5257): 520–21.
</div>
<div class="csl-entry">
———. 1972. <span>“<span class="nocase"><span>E</span>xtension of covariance selection mathematics</span>.”</span> <em>Ann Hum Genet</em> 35 (4): 485–90.
</div>
<div class="csl-entry">
R Core Team. 2021. <em>R: A Language and Environment for Statistical Computing</em>. Vienna, Austria: R Foundation for Statistical Computing. <a href="https://www.R-project.org/">https://www.R-project.org/</a>.
</div>
<div class="csl-entry">
Rice, Sean H. 2004. <em>Evolutionary Theory: Mathematical and Conceptual Foundations</em>. Sinauer Associates (Oxford University Press).
</div>
<div class="csl-entry">
Robertson, Alan. 1959. <span>“The Sampling Variance of the Genetic Correlation Coefficient.”</span> <em>Biometrics</em> 15 (3): 469–85.
</div>
<div class="csl-entry">
———. 1966. <span>“A Mathematical Model of the Culling Process in Dairy Cattle.”</span> <em>Animal Production</em> 8: 95–108.
</div>
<div class="csl-entry">
Robinson, D. Ashley, Daniel Falush, and Edward J Feil. 2010. <em>Bacterial Population Genetics in Infectious Disease</em>. Wiley-Blackwell.
</div>
<div class="csl-entry">
Rocha, E. P. C. 2018. <span>“<span class="nocase"><span>N</span>eutral <span>T</span>heory, <span>M</span>icrobial <span>P</span>ractice: <span>C</span>hallenges in <span>B</span>acterial <span>P</span>opulation <span>G</span>enetics</span>.”</span> <em>Mol Biol Evol</em> 35 (6): 1338–47.
</div>
<div class="csl-entry">
Rocha, Eduardo P C. 2004. <span>“Codon Usage Bias from tRNA’s Point of View: Redundancy, Specialization, and Efficient Decoding for Translation Optimization.”</span> <em>Genome Research</em> 14 (11): 2279–86. <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=525687&amp;tool=pmcentrez&amp;rendertype=abstract">http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=525687&amp;tool=pmcentrez&amp;rendertype=abstract</a>.
</div>
<div class="csl-entry">
Rocha, Eduardo P C, and Antoine Danchin. 2002. <span>“Base Composition Bias Might Result from Competition for Metabolic Resources.”</span> <em>Trends in Genetics</em> 18 (6): 291–94. <a href="http://www.ncbi.nlm.nih.gov/pubmed/12044357">http://www.ncbi.nlm.nih.gov/pubmed/12044357</a>.
</div>
<div class="csl-entry">
Rocha, Eduardo P C, and Edward J Feil. 2010. <span>“Mutational Patterns Cannot Explain Genome Composition: Are There Any Neutral Sites in the Genomes of Bacteria?”</span> Edited by Michael WEditor Nachman. <em>PLoS Genetics</em> 6 (9): 4. <a href="http://dx.doi.org/10.1371/journal.pgen.1001104">http://dx.doi.org/10.1371/journal.pgen.1001104</a>.
</div>
<div class="csl-entry">
Rogers, Anna R, Jeffrey C Dunne, Cinta Romay, Martin Bohn, Edward S Buckler, Ignacio A Ciampitti, Jode Edwards, et al. 2021. <span>“<span class="nocase">The importance of dominance and genotype-by-environment interactions on grain yield variation in a large-scale public cooperative maize experiment</span>.”</span> <em>G3 Genes|Genomes|Genetics</em> 11 (2). <a href="https://doi.org/10.1093/g3journal/jkaa050">https://doi.org/10.1093/g3journal/jkaa050</a>.
</div>
<div class="csl-entry">
Romero, Héctor, Emiliano Pereira, Hugo Naya, and Héctor Musto. 2009. <span>“Oxygen and Guanine-Cytosine Profiles in Marine Environments.”</span> <em>Journal of Molecular Evolution</em> 69 (2): 203–6.
</div>
<div class="csl-entry">
Rosenberg, N. A., and J. T. Kang. 2015. <span>“<span class="nocase"><span>G</span>enetic <span>D</span>iversity and <span>S</span>ocietally <span>I</span>mportant <span>D</span>isparities</span>.”</span> <em>Genetics</em> 201 (1): 1–12.
</div>
<div class="csl-entry">
Rosenberg, N. A., and M. Nordborg. 2002. <span>“<span class="nocase"><span>G</span>enealogical trees, coalescent theory and the analysis of genetic polymorphisms</span>.”</span> <em>Nat Rev Genet</em> 3 (5): 380–90.
</div>
<div class="csl-entry">
Roymondal, Uttam, Shibsankar Das, and Satyabrata Sahoo. 2009. <span>“Predicting Gene Expression Level from Relative Codon Usage Bias: An Application to Escherichia Coli Genome.”</span> <em>DNA Research An International Journal for Rapid Publication of Reports on Genes and Genomes</em> 16 (1): 13–30. <a href="http://www.ncbi.nlm.nih.gov/pubmed/19131380">http://www.ncbi.nlm.nih.gov/pubmed/19131380</a>.
</div>
<div class="csl-entry">
Rozen, S., H. Skaletsky, J. D. Marszalek, P. J. Minx, H. S. Cordum, R. H. Waterston, R. K. Wilson, and D. C. Page. 2003. <span>“<span class="nocase"><span>A</span>bundant gene conversion between arms of palindromes in human and ape <span>Y</span> chromosomes</span>.”</span> <em>Nature</em> 423 (6942): 873–76.
</div>
<div class="csl-entry">
Saccone, S, A De Sario, J Wiegant, A K Raap, G Della Valle, and G Bernardi. 1993. <span>“Correlations Between Isochores and Chromosomal Bands in the Human Genome.”</span> <em>Proceedings of the National Academy of Sciences of the United States of America</em> 90 (24): 11929–33. <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=48098&amp;tool=pmcentrez&amp;rendertype=abstract">http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=48098&amp;tool=pmcentrez&amp;rendertype=abstract</a>.
</div>
<div class="csl-entry">
Sánchez, A. L., J. I. Urioste, F. Peñagaricano, K. Neimaur, I. Sienra, H. Naya, and R. Kremer. 2016. <span>“<span class="nocase">Genetic parameters of objectionable fibers and of their associations with fleece traits in Corriedale sheep1</span>.”</span> <em>Journal of Animal Science</em> 94 (1): 13–20. <a href="https://doi.org/10.2527/jas.2015-9619">https://doi.org/10.2527/jas.2015-9619</a>.
</div>
<div class="csl-entry">
Sardell, J. M., and M. Kirkpatrick. 2020. <span>“<span class="nocase"><span>S</span>ex <span>D</span>ifferences in the <span>R</span>ecombination <span>L</span>andscape</span>.”</span> <em>Am Nat</em> 195 (2): 361–79.
</div>
<div class="csl-entry">
Schmidt, P., J. Hartung, J. Rath, and H.-P. Piepho. 2019. <span>“Estimating Broad-Sense Heritability with Unbalanced Data from Agricultural Cultivar Trials.”</span> <em>Crop Science</em> 59: 525–36.
</div>
<div class="csl-entry">
Searle, Shayle R. 1961. <span>“Phenotypic, Genetic and Environmental Correlations.”</span> <em>Biometrics</em> 17 (3): 474–80.
</div>
<div class="csl-entry">
Sequencing, The Chimpanzee, and Analysis Consortium. 2005. <span>“Initial Sequence of the Chimpanzee Genome and Comparison with the Human Genome.”</span> <em>Nature</em> 437 (7055): 69–87. <a href="http://www.ncbi.nlm.nih.gov/pubmed/16136131">http://www.ncbi.nlm.nih.gov/pubmed/16136131</a>.
</div>
<div class="csl-entry">
Sharp, P M, and W H Li. 1987. <span>“The Codon Adaptation Index–a Measure of Directional Synonymous Codon Usage Bias, and Its Potential Applications.”</span> <em>Nucleic Acids Research</em> 15 (3): 1281–95. <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=3547335">http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=3547335</a>.
</div>
<div class="csl-entry">
Sharp, P M, T M Tuohy, and K R Mosurski. 1986. <span>“Codon Usage in Yeast: Cluster Analysis Clearly Differentiates Highly and Lowly Expressed Genes.”</span> <em>Nucleic Acids Research</em> 14 (13): 5125–43. <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=311530&amp;tool=pmcentrez&amp;rendertype=abstract">http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=311530&amp;tool=pmcentrez&amp;rendertype=abstract</a>.
</div>
<div class="csl-entry">
Soppa, J. 2014. <span>“<span class="nocase"><span>P</span>olyploidy in archaea and bacteria: about desiccation resistance, giant cell size, long-term survival, enforcement by a eukaryotic host and additional aspects</span>.”</span> <em>J Mol Microbiol Biotechnol</em> 24 (5-6): 409–19.
</div>
<div class="csl-entry">
Sorensen, Daniel, and Daniel Gianola. 2002. <em>Likelihood, Bayesian, and MCMC Methods in Quantitative Genetics</em>. Springer-Verlag.
</div>
<div class="csl-entry">
Spangenberg, L, F Battke, M Graña, K Nieselt, and H Naya. 2011. <span>“Identifying Associations Between Amino Acid Changes and Meta Information in Alignments.”</span> <em>Bioinformatics</em> 27 (20): 2782–89. <a href="http://www.ncbi.nlm.nih.gov/pubmed/21846738">http://www.ncbi.nlm.nih.gov/pubmed/21846738</a>.
</div>
<div class="csl-entry">
Spangenberg, L., A. Correa, B. Dallagiovanna, and H. Naya. 2013. <span>“<span class="nocase"><span>R</span>ole of alternative polyadenylation during adipogenic differentiation: an in silico approach</span>.”</span> <em>PLoS One</em> 8 (10): e75578.
</div>
<div class="csl-entry">
Stein, J. L., T. L. Marsh, K. Y. Wu, H. Shizuya, and E. F. DeLong. 1996. <span>“<span class="nocase"><span>C</span>haracterization of uncultivated prokaryotes: isolation and analysis of a 40-kilobase-pair genome fragment from a planktonic marine archaeon</span>.”</span> <em>J Bacteriol</em> 178 (3): 591–99.
</div>
<div class="csl-entry">
Sueoka, Noboru. 1962. <span>“ON THE GENETIC BASIS OF VARIATION AND HETEROGENEITY OF DNA BASE COMPOSITION.”</span> <em>Proceedings of the National Academy of Sciences of the United States of America</em> 48 (4): 582–92. <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC220819/">http://www.ncbi.nlm.nih.gov/pmc/articles/PMC220819/</a>.
</div>
<div class="csl-entry">
Tagliani-Ribeiro, A., M. Oliveira, A. K. Sassi, M. R. Rodrigues, M. Zagonel-Oliveira, G. Steinman, U. Matte, N. J. Fagundes, and L. Schuler-Faccini. 2011. <span>“<span class="nocase"><span>T</span>win <span>T</span>own in <span>S</span>outh <span>B</span>razil: a <span>N</span>azi’s experiment or a genetic founder effect?</span>”</span> <em>PLoS One</em> 6 (6): e20328.
</div>
<div class="csl-entry">
Tajima, F. 1983. <span>“Evolutionary Relationship of DNA Sequences in Finite Populations.”</span> <em>Genetics</em> 105 (2): 437–60.
</div>
<div class="csl-entry">
Tallis, George Michael. 1959. <span>“SAMPLING ERRORS OF GENETIC CORRELATION COEFFICIENTS CALCULATED FROM ANALYSES o f VARIANCE AND COVARIANCE.”</span> <em>Australian Journal of Statistics</em> 1 (2): 35–43.
</div>
<div class="csl-entry">
Thomsen, B., P. Horn, F. Panitz, E. Bendixen, A. H. Petersen, L. E. Holm, V. H. Nielsen, J. S. Agerholm, J. Arnbjerg, and C. Bendixen. 2006. <span>“<span class="nocase"><span>A</span> missense mutation in the bovine <span>S</span><span>L</span><span>C</span>35<span>A</span>3 gene, encoding a <span>U</span><span>D</span><span>P</span>-<span>N</span>-acetylglucosamine transporter, causes complex vertebral malformation</span>.”</span> <em>Genome Res</em> 16 (1): 97–105.
</div>
<div class="csl-entry">
Thurman, T. J., and R. D. Barrett. 2016. <span>“<span class="nocase"><span>T</span>he genetic consequences of selection in natural populations</span>.”</span> <em>Mol Ecol</em> 25 (7): 1429–48.
</div>
<div class="csl-entry">
Van Inghelandt, D., J. C. Reif, B. S. Dhillon, P. Flament, and A. E. Melchinger. 2011. <span>“<span class="nocase"><span>E</span>xtent and genome-wide distribution of linkage disequilibrium in commercial maize germplasm</span>.”</span> <em>Theor Appl Genet</em> 123 (1): 11–20.
</div>
<div class="csl-entry">
VanRaden, P. M. 2008. <span>“<span class="nocase"><span>E</span>fficient methods to compute genomic predictions</span>.”</span> <em>J Dairy Sci</em> 91 (11): 4414–23.
</div>
<div class="csl-entry">
Visscher, P. M., W. G. Hill, and N. R. Wray. 2008. <span>“<span class="nocase"><span>H</span>eritability in the genomics era–concepts and misconceptions</span>.”</span> <em>Nat Rev Genet</em> 9 (4): 255–66.
</div>
<div class="csl-entry">
Vries, H. de. 1900. <span>“<span class="nocase"><span>S</span>ur la loi de disjonction des Hybrides</span>.”</span> <em>Comptes Rendus de l‘Acad<span>é</span>mie Des Sciences</em> 130 (-): 845–47.
</div>
<div class="csl-entry">
Waddington, Conrad Hal. 1942. <span>“Canalization of Development and the Inheritance of Acquired Characters.”</span> <em>Nature</em> 150 (3811): 563–65.
</div>
<div class="csl-entry">
Wahlund, Sten. 1928. <span>“Zusammensetzung von Populationen Und Korrelationserscheinungen Vom Standpunkt Der Vererbungslehre Aus Betrachtet.”</span> <em>Hereditas</em> 11 (1): 65–106.
</div>
<div class="csl-entry">
Walsh, Bruce, and Michael Lynch. 2018. <em>Evolution and Selection of Quantitative Traits</em>. Oxford University Press.
</div>
<div class="csl-entry">
Weinberg, W. 1908. <span>“<span class="nocase"><span>Ü</span>ber den Nachweis der Vererbung beim Menschen</span>.”</span> <em>Jahreshefte Des Vereins f<span>ü</span>r Vaterl<span>ä</span>ndische Naturkunde in W<span>ü</span>rttemberg.</em> 64: 369–82.
</div>
<div class="csl-entry">
Werf, Julius van der. 2000. <span>“Genetic Evaluation and Breeding Program Design.”</span> <a href="https://jvanderw.une.edu.au/gene422.htm">https://jvanderw.une.edu.au/gene422.htm</a>.
</div>
<div class="csl-entry">
Woese, C. R., O. Kandler, and M. L. Wheelis. 1990. <span>“<span class="nocase"><span>T</span>owards a natural system of organisms: proposal for the domains <span>A</span>rchaea, <span>B</span>acteria, and <span>E</span>ucarya</span>.”</span> <em>Proc Natl Acad Sci U S A</em> 87 (12): 4576–79.
</div>
<div class="csl-entry">
Woltereck, Richard. 1909. <span>“Weitere Experimentelle Untersuchungen Uber Artveranderung, Speziell Uber Das Wesen Quantitativer Artunterschiede Bei Daphniden.”</span> <em>Verhandlungen Der Deutschen Zoologischen Gesellschaft Leipzig, 19 110-173</em> 19: 110–73.
</div>
<div class="csl-entry">
Wooldrigde, Jeffrey M. 2013. <em>Introductory Econometrics: A Modern Approach</em>. 5th. ed. South-Western, Cengage Learning.
</div>
<div class="csl-entry">
Wright, D. 2015. <span>“<span class="nocase"><span>T</span>he <span>G</span>enetic <span>A</span>rchitecture of <span>D</span>omestication in <span>A</span>nimals</span>.”</span> <em>Bioinform Biol Insights</em> 9 (Suppl 4): 11–20.
</div>
<div class="csl-entry">
Wright, F. 1990. <span>“The <span>‘Effective Number of Codons’</span> Used in a Gene.”</span> <em>Gene</em> 87 (1): 23–29. <a href="http://www.ncbi.nlm.nih.gov/pubmed/2110097">http://www.ncbi.nlm.nih.gov/pubmed/2110097</a>.
</div>
<div class="csl-entry">
Wright, S. 1918. <span>“<span class="nocase"><span>O</span>n the <span>N</span>ature of <span>S</span>ize <span>F</span>actors</span>.”</span> <em>Genetics</em> 3 (4): 367–74.
</div>
<div class="csl-entry">
———. 1921. <span>“<span class="nocase"><span>S</span>ystems of <span>M</span>ating. <span>I</span>. the <span>B</span>iometric <span>R</span>elations between <span>P</span>arent and <span>O</span>ffspring</span>.”</span> <em>Genetics</em> 6 (2): 111–23.
</div>
<div class="csl-entry">
———. 1932. <span>“<span class="nocase"><span>T</span>he roles of mutation, inbreeding, cross- breeding and selection in evolution</span>.”</span> <em>Proceedings of the Sixth Annual Congress of Genetics</em> 1 (-): 356–66.
</div>
<div class="csl-entry">
———. 1937. <span>“<span class="nocase"><span>T</span>he <span>D</span>istribution of <span>G</span>ene <span>F</span>requencies in <span>P</span>opulations</span>.”</span> <em>Proc Natl Acad Sci U S A</em> 23 (6): 307–20.
</div>
<div class="csl-entry">
———. 1943. <span>“<span class="nocase"><span>G</span>enetics <span>I</span>solation by <span>D</span>istance</span>.”</span> <em>Genetics</em> 28 (2): 114–38.
</div>
<div class="csl-entry">
WRIGHT, S. 1951. <span>“<span class="nocase"><span>A</span>nn <span>E</span>ugen<span>T</span>he genetical structure of populations</span>.”</span> <em>Ann Eugen</em> 15 (4): 323–54.
</div>
<div class="csl-entry">
Wright, Sewall. 1922. <span>“<span class="nocase">Coefficients of inbreeding and relationship</span>.”</span> <em>American Naturalist</em> 56: 330–38.
</div>
<div class="csl-entry">
———. 1934a. <span>“AN ANALYSIS OF VARIABILITY IN NUMBER OF DIGITS IN AN INBRED STRAIN OF GUINEA PIGS.”</span> <em>Genetics</em> 19 (6): 506–36.
</div>
<div class="csl-entry">
———. 1934b. <span>“The Method of Path Coefficients.”</span> <em>Ann. Math. Statist.</em> 5 (3): 161–215.
</div>
<div class="csl-entry">
———. 1978. <em>Evolution and the Genetics of Populations</em>. Chicago Press.
</div>
<div class="csl-entry">
Yakovchuk, Peter, Ekaterina Protozanova, and Maxim D Frank-Kamenetskii. 2006. <span>“Base-Stacking and Base-Pairing Contributions into Thermal Stability of the DNA Double Helix.”</span> <em>Nucleic Acids Research</em> 34 (2): 564–74. <a href="http://www.ncbi.nlm.nih.gov/pubmed/16449200">http://www.ncbi.nlm.nih.gov/pubmed/16449200</a>.
</div>
<div class="csl-entry">
Yasuda, N. 1984. <span>“<span class="nocase"><span>A</span> note on gene frequency estimation in the <span>A</span><span>B</span><span>O</span> and <span>A</span><span>B</span><span>O</span>-like system</span>.”</span> <em>Jinrui Idengaku Zasshi</em> 29 (3): 371–80.
</div>
<div class="csl-entry">
Yengo, L., N. R. Wray, and P. M. Visscher. 2021. <span>“Extreme Inbreeding in a European Ancestry Sample from the Contemporary UK Population.”</span> <em>Nature Communications</em> 10 (3719).
</div>
<div class="csl-entry">
Yue, X. P., C. Dechow, and W. S. Liu. 2015. <span>“<span class="nocase"><span>A</span> limited number of <span>Y</span> chromosome lineages is present in <span>N</span>orth <span>A</span>merican <span>H</span>olsteins</span>.”</span> <em>J Dairy Sci</em> 98 (4): 2738–45.
</div>
<div class="csl-entry">
Yule, George Udny. 1902. <span>“MENDEL’s LAWS AND THEIR PROBABLE RELATIONS TO INTRA-RACIAL HEREDITY.”</span> <em>New Phytologist</em> 1: 193–207, 222–38.
</div>
<div class="csl-entry">
Zerial, M, J Salinas, J Filipski, and G Bernardi. 1986. <span>“Gene Distribution and Nucleotide Sequence Organization in the Human Genome.”</span> <em>The Federation of European Biochemical Societies Journal</em> 160 (3): 479–85. <a href="http://www.ncbi.nlm.nih.gov/pubmed/3780716">http://www.ncbi.nlm.nih.gov/pubmed/3780716</a>.
</div>
<div class="csl-entry">
Zwald, N. R., K. A. Weigel, W. F. Fikse, and R. Rekaya. 2003. <span>“<span class="nocase"><span>I</span>dentification of factors that cause genotype by environment interaction between herds of <span>H</span>olstein cattle in seventeen countries</span>.”</span> <em>J Dairy Sci</em> 86 (3): 1009–18.
</div>
<div class="csl-entry">
Zwietering, M. H., I. Jongenburger, F. M. Rombouts, and K. van ’t Riet. 1990. <span>“<span class="nocase"><span>M</span>odeling of the bacterial growth curve</span>.”</span> <em>Appl Environ Microbiol</em> 56 (6): 1875–81.
</div>
</div>
</div>





































































































































            </section>

          </div>
        </div>
      </div>
<a href="apendicedos.html" class="navigation navigation-prev navigation-unique" aria-label="Previous page"><i class="fa fa-angle-left"></i></a>

    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/rstudio/bookdown-demo/edit/master/18-references.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["ApuntesGeneticaII.pdf", "ApuntesGeneticaII.epub"],
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.9/latest.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
